<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="Benjamin J. Heil" />
  <meta name="author" content="Casey S. Greene" />
  <meta name="dcterms.date" content="2022-12-05" />
  <meta name="keywords" content="citation metrics, pagerank, science of science" />
  <title>The field-dependent nature of PageRank values in citation networks</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    span.underline{text-decoration: underline;}
    div.column{display: inline-block; vertical-align: top; width: 50%;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
    }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
  <!--
  Manubot generated metadata rendered from header-includes-template.html.
  Suggest improvements at https://github.com/manubot/manubot/blob/main/manubot/process/header-includes-template.html
  -->
  <meta name="dc.format" content="text/html" />
  <meta name="dc.title" content="The field-dependent nature of PageRank values in citation networks" />
  <meta name="citation_title" content="The field-dependent nature of PageRank values in citation networks" />
  <meta property="og:title" content="The field-dependent nature of PageRank values in citation networks" />
  <meta property="twitter:title" content="The field-dependent nature of PageRank values in citation networks" />
  <meta name="dc.date" content="2022-12-05" />
  <meta name="citation_publication_date" content="2022-12-05" />
  <meta name="dc.language" content="en-US" />
  <meta name="citation_language" content="en-US" />
  <meta name="dc.relation.ispartof" content="Manubot" />
  <meta name="dc.publisher" content="Manubot" />
  <meta name="citation_journal_title" content="Manubot" />
  <meta name="citation_technical_report_institution" content="Manubot" />
  <meta name="citation_author" content="Benjamin J. Heil" />
  <meta name="citation_author_institution" content="Genomics and Computational Biology Graduate Group, Perelman School of Medicine, University of Pennsylvania" />
  <meta name="citation_author_orcid" content="0000-0002-2811-1031" />
  <meta name="twitter:creator" content="@autobencoder" />
  <meta name="citation_author" content="Casey S. Greene" />
  <meta name="citation_author_institution" content="Department of Pharmacology, University of Colorado School of Medicine" />
  <meta name="citation_author_institution" content="Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine" />
  <meta name="citation_author_orcid" content="0000-0001-8713-9213" />
  <meta name="twitter:creator" content="@greenescientist" />
  <link rel="canonical" href="https://greenelab.github.io/indices_manuscript/" />
  <meta property="og:url" content="https://greenelab.github.io/indices_manuscript/" />
  <meta property="twitter:url" content="https://greenelab.github.io/indices_manuscript/" />
  <meta name="citation_fulltext_html_url" content="https://greenelab.github.io/indices_manuscript/" />
  <meta name="citation_pdf_url" content="https://greenelab.github.io/indices_manuscript/manuscript.pdf" />
  <link rel="alternate" type="application/pdf" href="https://greenelab.github.io/indices_manuscript/manuscript.pdf" />
  <link rel="alternate" type="text/html" href="https://greenelab.github.io/indices_manuscript/v/973020713e98d2e14242958c6997ae0aef37c803/" />
  <meta name="manubot_html_url_versioned" content="https://greenelab.github.io/indices_manuscript/v/973020713e98d2e14242958c6997ae0aef37c803/" />
  <meta name="manubot_pdf_url_versioned" content="https://greenelab.github.io/indices_manuscript/v/973020713e98d2e14242958c6997ae0aef37c803/manuscript.pdf" />
  <meta property="og:type" content="article" />
  <meta property="twitter:card" content="summary_large_image" />
  <link rel="icon" type="image/png" sizes="192x192" href="https://manubot.org/favicon-192x192.png" />
  <link rel="mask-icon" href="https://manubot.org/safari-pinned-tab.svg" color="#ad1457" />
  <meta name="theme-color" content="#ad1457" />
  <!-- end Manubot generated metadata -->
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
</head>
<body>
<header id="title-block-header">
<h1 class="title">The field-dependent nature of PageRank values in citation networks</h1>
</header>
<p><small><em>
This manuscript
(<a href="https://greenelab.github.io/indices_manuscript/v/973020713e98d2e14242958c6997ae0aef37c803/">permalink</a>)
was automatically generated
from <a href="https://github.com/greenelab/indices_manuscript/tree/973020713e98d2e14242958c6997ae0aef37c803">greenelab/indices_manuscript@9730207</a>
on December 5, 2022.
</em></small></p>
<h2 id="authors">Authors</h2>
<ul>
<li><p><strong>Benjamin J. Heil</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-2811-1031">0000-0002-2811-1031</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/ben-heil">ben-heil</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/autobencoder">autobencoder</a><br>
<small>
Genomics and Computational Biology Graduate Group, Perelman School of Medicine, University of Pennsylvania
· Funded by The Gordon and Betty Moore Foundation (GBMF4552)
</small></p></li>
<li><p><strong>Casey S. Greene</strong>
<sup><a href="#correspondence">✉</a></sup><br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-8713-9213">0000-0001-8713-9213</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/cgreene">cgreene</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/greenescientist">greenescientist</a><br>
<small>
Department of Pharmacology, University of Colorado School of Medicine; Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine
· Funded by The Gordon and Betty Moore Foundation (GBMF4552); The National Human Genome Research Institute (R01 HG10067)
</small></p></li>
</ul>
<div id="correspondence">
<p>✉ — Correspondence possible via <a href="https://github.com/greenelab/indices_manuscript/issues">GitHub Issues</a>
or email to
Casey S. Greene &lt;casey.s.greene@cuanschutz.edu&gt;.</p>
</div>
<h2 class="page_break_before" id="abstract">Abstract</h2>
<p>The value of scientific research can be easier to assess at the collective level than at the level of individual contributions.
Several journal-level and article-level metrics aim to measure the importance of journals or individual manuscripts.
However, many are citation-based and citation practices between fields.
To account for these differences, scientists have devised normalization schemes to make metrics more comparable across fields.
We use PageRank as an example metric and examine the extent to which field-specific citation norms drive estimated importance differences.
We recapitulate differences in journal and article PageRanks between fields.
We find that manuscripts shared between fields have different PageRanks depending on which field’s citation network the metric is calculated in.
We implement a degree-preserving graph shuffling algorithm to generate a null distribution of similar networks and find differences more likely attributed to field-specific preferences than citation patterns.
While differences exist between fields’ metric distributions, applying metrics in a field-aware manner rather than using normalized global metrics avoids losing important information about article preferences.
Our results also suggest that assigning a single importance value to a manuscript may not be a useful construct, as the importance of each manuscript varies by the reader’s field.</p>
<h2 id="introduction">Introduction</h2>
<p>There are more academic papers than any human can read in a lifetime.
Attention has been given to ranking papers, journals, or researchers by their “importance,” assessed via various metrics.
Citation count assumes the number of citations determines a paper’s importance.
The h-index and Journal Impact Factor focus on secondary factors like author or journal track records.
Graph-based methods like PageRank or disruption index use the context of the citing papers to evaluate an article’s relevance <span class="citation" data-cites="qXsETMbA 16V0td5f2 zeuTsDVX UTLLCTKy">[<a href="#ref-qXsETMbA" role="doc-biblioref">1</a>,<a href="#ref-16V0td5f2" role="doc-biblioref">2</a>,<a href="#ref-zeuTsDVX" role="doc-biblioref">3</a>,<a href="#ref-UTLLCTKy" role="doc-biblioref">4</a>]</span>.
Each of these methods has limitations, and permutations exist that attempt to shore up specific weaknesses <span class="citation" data-cites="kZI40ZXS 11IkPxdIL 12CYfIcWF PGRGcmLi">[<a href="#ref-kZI40ZXS" role="doc-biblioref">5</a>,<a href="#ref-11IkPxdIL" role="doc-biblioref">6</a>,<a href="#ref-12CYfIcWF" role="doc-biblioref">7</a>,<a href="#ref-PGRGcmLi" role="doc-biblioref">8</a>]</span>.</p>
<p>One objection to such practices is that “importance” is subjective.
The San Francisco Declaration on Research Assessment (DORA) argues against using Journal Impact Factor, or any journal-based metric, to assess individual manuscripts or scientists <span class="citation" data-cites="62Q1rmv4">[<a href="#ref-62Q1rmv4" role="doc-biblioref">9</a>]</span>.
DORA further argues in favor of evaluating the content of scientific content of articles and notes that any metrics used should be article-level ones (https://sfdora.org/read/).
However, even article-level importance ignores that the importance of a specific scientific output will fundamentally differ across fields.
Nobel prize-winning work will often be less important in cancer biology if the prize-winning article is an astrophysics one.</p>
<p>Because there are differences between fields’ citation practices <span class="citation" data-cites="Hw4ZDEnz">[<a href="#ref-Hw4ZDEnz" role="doc-biblioref">10</a>]</span>, scientists have developed strategies including normalizing the number of citations based on nearby papers in a citation network, rescaling fields’ citation data to give more consistent PageRank results, and so on <span class="citation" data-cites="kZI40ZXS 7n76ZZPt 3FJspt0W 15JxaxOiR">[<a href="#ref-kZI40ZXS" role="doc-biblioref">5</a>,<a href="#ref-7n76ZZPt" role="doc-biblioref">11</a>,<a href="#ref-3FJspt0W" role="doc-biblioref">12</a>,<a href="#ref-15JxaxOiR" role="doc-biblioref">13</a>]</span>.
Such approaches normalize away field-specific effects, which might help to compare one researcher with another in a very different field but do not address the difference in the relevance of a topic between fields.
This phenomenon of field-specific importance has been observed at the level of journal metrics.
Mason and Singh recently noted that depending on the field, the journal <em>Christian Higher Education</em> is either ranked as a Q1 (top quartile) journal or a Q4 (bottom quartile) journal <span class="citation" data-cites="AkQLg7kt">[<a href="#ref-AkQLg7kt" role="doc-biblioref">14</a>]</span>.</p>
<p>It is possible that, while global journal-level metrics fail to capture field-specific importance, article-level metrics are sufficiently granular that the importance of a manuscript remains constant across fields.
We sought to examine the extent to which article-level metrics generalize between fields.
We examine this using MeSH terms to define fields and use field-specific citation graphs to assess their importance within the field.
While it is trivially apparent that journals or articles that do not have cross-field citations will have variable importance, we ignore these cases.
We include only those including citations in both fields, where we expect possible consistency.
We first replicate previous findings that journal-level metrics can differ substantially among fields.
We also find field-specific variability in importance at the article level.
We make our results explorable through a web app that shows metrics for overlapping papers between pairs of fields.</p>
<p>Our results show that even article-level metrics can differ substantially among fields.
We recommend that metrics used for assessing research outputs include field-specific, in addition to global, ones.
While qualitative assessment of the content of manuscripts remains time-consuming, our results suggest that within-field and across-field assessment remains key to assessing the importance of research outputs.</p>
<h2 id="results">Results</h2>
<h3 id="journal-rankings-differ-between-fields">Journal rankings differ between fields</h3>
<p>In an attempt to quantify the relative importance of journals, scientists have created rankings using metrics the Journal Impact Factor, which essentially uses citations per article, and those that rely on more complex representations like Eigenfactor <span class="citation" data-cites="4TkOo7Oy">[<a href="#ref-4TkOo7Oy" role="doc-biblioref">15</a>]</span>.
Previous reports note that journal rankings differ substantially between fields using metrics based on citation numbers <span class="citation" data-cites="AkQLg7kt">[<a href="#ref-AkQLg7kt" role="doc-biblioref">14</a>]</span>.
We calculated a field-specific PageRank-based score for each journal as the median PageRank of manuscripts published in that journal for that field (Fig. <a href="#fig:journal">1</a> A).
We first sought to understand the extent to which PageRank replicated journal ranking differences across fields.</p>
<p>To begin, we compared the differences in ranking between the top fifty journals in nanotechnology and their corresponding ranks in microscopy.
While the ranks were correlated (TODO:CORRELATION STAT) there was a great deal of variance, especially for journals outside the top 20 in nanotechnology (Fig. <a href="#fig:journal">1</a> B).
We then examined the top-ranked journal in each of our 45 fields to determine whether the top-ranking journal was consistent across fields (Fig. <a href="#fig:journal">1</a> C).
We found that the most commonly top-ranked journal was <em>Science</em>.
This was unsurprising, given that it tends to rank highly among global journal-level metrics such as eigenfactor.
However, while <em>Science</em> was the top-ranked journal in a plurality of fields, approximately 80% of fields had a different journal in that spot.</p>
<p>We also investigated the presence of single-topic journals in our dataset, as MeSH headings reflect a different type of aggregation than journals do <span class="citation" data-cites="ErD8Jjn8">[<a href="#ref-ErD8Jjn8" role="doc-biblioref">16</a>]</span>.
Of the 5,178 journals with at least 50 articles in our dataset, the median number of fields publishing in a given journal is 15 (Fig. <a href="#fig:journal">1</a> D).
In the context of MeSH, specialty journals are rare.
Most journals publish manuscripts with in one-third or more of the MeSH headings in our dataset.</p>
<div id="fig:journal" class="fignos">
<figure>
<img src="./images/journal_fig.png" style="width:100.0%" alt="Figure 1: Journals’ PageRank-derived rankings differ between fields. A) A schematic showing how paired networks are derived from the full citation network. B) A comparison of the ranks of the top 50 journals by PageRank in nanotechnology and their rank in microscopy. Top-50 nanotechnology journals with no papers in microscopy have been omitted. C) The frequency with which journals in the dataset are the top journal for a field. D) The distribution of fields published per journal. The X-axis corresponds to the number of fields for which a journal has at least one paper within the field. All plots restrict the set of journals to those with at least 50 papers in the dataset." />
<figcaption aria-hidden="true"><span>Figure 1:</span> Journals’ PageRank-derived rankings differ between fields.
A) A schematic showing how paired networks are derived from the full citation network.
B) A comparison of the ranks of the top 50 journals by PageRank in nanotechnology and their rank in microscopy.
Top-50 nanotechnology journals with no papers in microscopy have been omitted.
C) The frequency with which journals in the dataset are the top journal for a field.
D) The distribution of fields published per journal.
The X-axis corresponds to the number of fields for which a journal has at least one paper within the field.
All plots restrict the set of journals to those with at least 50 papers in the dataset.</figcaption>
</figure>
</div>
<h3 id="manuscript-pageranks-differ-between-fields">Manuscript PageRanks differ between fields</h3>
<p>We split the citation network into its component fields and calculated the PageRank for each article (Fig. <a href="#fig:distribution">2</a> A).
We examined the distribution of PageRanks across fields and found that they differed greatly (Fig. <a href="#fig:distribution">2</a> B).
We first examined whether the citation practices of fields contributed to importance differences.
Investigating manuscripts that appeared in pairs of fields, we found that the distribution of importances matched the network more than that of the alternative topic area of the manuscript (Fig. <a href="#fig:distribution">2</a> B, C).
Given the differences in distributions in articles shared by these fields (Fig. <a href="#fig:distribution">2</a> D), we found it difficult to determine whether correspondence between fields was random or due to different degrees of interest in certain articles (Fig. <a href="#fig:distribution">2</a> E). <!-- I am not sure I get what you're trying to say here about E - aren't you going to go into this for the next section? I also don't get how the ECDF is getting you to random or not. What about dropping panel E here? It feels like that might be the cleanest way to avoid having to get into things too early and confusing your reader. --></p>
<div id="fig:distribution" class="fignos">
<figure>
<img src="./images/distribution_fig.png" style="width:100.0%" alt="Figure 2: Differences in the distribution of PageRanks between fields. A) A schematic showing how field pairs are split and their PageRanks are calculated. B) The distribution of article PageRanks for nanotechnology and microscopy. The distributions marked with ‘All’ contain all the papers for the given field in the dataset, while those marked ‘overlapping’ contain only articles present in both fields. C) The empirical cumulative density functions of nanotechnology and microscopy. D) The differences in distribution of the PageRanks of articles shared by nanotechnology and microscopy. E) A density plot showing the joint distribution of PageRanks for papers overlapping in nanotechnology and microscopy." />
<figcaption aria-hidden="true"><span>Figure 2:</span> Differences in the distribution of PageRanks between fields.
A) A schematic showing how field pairs are split and their PageRanks are calculated.
B) The distribution of article PageRanks for nanotechnology and microscopy.
The distributions marked with ‘All’ contain all the papers for the given field in the dataset, while those marked ‘overlapping’ contain only articles present in both fields.
C) The empirical cumulative density functions of nanotechnology and microscopy.
D) The differences in distribution of the PageRanks of articles shared by nanotechnology and microscopy.
E) A density plot showing the joint distribution of PageRanks for papers overlapping in nanotechnology and microscopy.</figcaption>
</figure>
</div>
<h3 id="fields-differences-are-not-solely-driven-by-differences-in-citation-practices">Fields’ differences are not solely driven by differences in citation practices</h3>
<p>We devised a strategy to generate an empirical null for a field pair under the assumption that the field pair represented a single, homogenous field (Fig. <a href="#fig:percentile">3</a> A).
For each field-pair intersection, we performed a degree-distribution preserving permutation.
We created 100 permuted networks for each field pair.
We then split the networks into their constituent fields and calculated a percentile using the number of permuted networks with a lower PageRank for a manuscript than the true PageRank.
A manuscript with a PageRank higher than all networks has a percentile of 100, and one lower than all permuted networks has a percentile of zero.
We used the difference in the percentile in each field as the field-specific affinity for a given paper.
This percentile score allowed us to control for the differing degree distributions between fields by comparing papers based on their expected PageRank in a random network with the same node degrees.</p>
<p>We selected field pairs with varying degrees of correlation between their PageRanks (Fig. <a href="#fig:percentile">3</a> B).
By examining the fields’ PageRank percentiles, we found that many articles had large differences in their perception between fields (Fig. <a href="#fig:percentile">3</a> C).
In nanotechnology and microscopy, papers with high nanotechnology percentiles and low microscopy percentiles tended towards applications of nanotechnology, while their counterparts with high microscopy percentiles and low nanotechnology percentiles were often papers about technological developments in microscopy (Fig. <a href="#fig:percentile">3</a> A, Table 1).
Immunochemistry-favored papers are largely applications of immunochemical methods, while anatomy-favored articles tend to focus experiments on a single anatomical region (Fig. <a href="#fig:percentile">3</a> B, Table 2).
Proteomics and metabolomics tend to use similar methods, so the fields on either end are largely (though not entirely) field-specific applications of those methods (Fig. <a href="#fig:percentile">3</a> C, Table 3).
Manuscripts favored in computational biology were similarly applications-focused.
However, those with more importance in human genetics tended towards policy papers due to its MeSH heading (H01.158.273.343.385) excluding fields like genomics, population genetics, and microbial genetics (Fig. <a href="#fig:percentile">3</a> D, Table 4).
In addition to papers with large differences between fields, each field pair has papers with high PageRanks and similar percentiles.
While some papers may be influential in multiple fields, others have more field-specific import.</p>
<p>It is impossible to describe all the field pairs and relevant differences between fields within the space of a journal article.
We have developed a web server that displays the percentiles for all pairs of fields in our dataset with at least 1000 shared articles (Fig. <a href="#fig:percentile">3</a> D), which can be accessed at TODO.
We hope that the availability of the web server and the reproducibility of our code will assist other scientists in uncovering new insights from this dataset.</p>
<div id="fig:percentile" class="fignos">
<figure>
<img src="./images/percentile_figure.png" style="width:100.0%" alt="Figure 3: Field-specific preferences in papers. A) A schematic showing how networks are shuffled and how articles’ percentile scores are calculated. The histograms at the bottom of the figure correspond to the distribution of PageRanks for the shuffled networks, while the red lines correspond to an article’s PageRank in the true citation network. B) The Pearson correlation of PageRanks between fields. The red points are the field pairs expanded in panel C. C) The percentile scores and PageRanks for overlapping articles in various fields. Points are colored based on the difference in percentile scores in the fields e.g. “Nanotechnology-Microscopy” corresponds to the difference between the nanotechnology and microscopy percentile scores. The numbers next to points are the reference number for the article in the bibliography. D) A screenshot of the webserver showing the percentile score difference and journal median PageRank plot functionality." />
<figcaption aria-hidden="true"><span>Figure 3:</span> Field-specific preferences in papers.
A) A schematic showing how networks are shuffled and how articles’ percentile scores are calculated.
The histograms at the bottom of the figure correspond to the distribution of PageRanks for the shuffled networks, while the red lines correspond to an article’s PageRank in the true citation network.
B) The Pearson correlation of PageRanks between fields.
The red points are the field pairs expanded in panel C.
C) The percentile scores and PageRanks for overlapping articles in various fields.
Points are colored based on the difference in percentile scores in the fields e.g. “Nanotechnology-Microscopy” corresponds to the difference between the nanotechnology and microscopy percentile scores.
The numbers next to points are the reference number for the article in the bibliography.
D) A screenshot of the webserver showing the percentile score difference and journal median PageRank plot functionality.</figcaption>
</figure>
</div>
<table>
<colgroup>
<col style="width: 25%" />
<col style="width: 25%" />
<col style="width: 25%" />
<col style="width: 25%" />
</colgroup>
<thead>
<tr class="header">
<th>Nanotechnology Percentile</th>
<th>Microscopy Percentile</th>
<th>Title</th>
<th>Reference</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>100</td>
<td>4</td>
<td>A robust DNA mechanical device controlled by hybridization topology</td>
<td><span class="citation" data-cites="1GphepDNl">[<a href="#ref-1GphepDNl" role="doc-biblioref">17</a>]</span></td>
</tr>
<tr class="even">
<td>100</td>
<td>5</td>
<td>Bioadhesive poly(methyl methacrylate) microdevices for controlled drug delivery</td>
<td><span class="citation" data-cites="NmOwHa9H">[<a href="#ref-NmOwHa9H" role="doc-biblioref">18</a>]</span></td>
</tr>
<tr class="odd">
<td>99</td>
<td>2</td>
<td>DNA-templated self-assembly of protein arrays and highly conductive nanowrires</td>
<td><span class="citation" data-cites="IkV2K836">[<a href="#ref-IkV2K836" role="doc-biblioref">19</a>]</span></td>
</tr>
<tr class="even">
<td>0</td>
<td>100</td>
<td>Photostable luminescent nanoparticles as biological label for cell recognition of system lupus erythematosus patients</td>
<td><span class="citation" data-cites="19ryujP9j">[<a href="#ref-19ryujP9j" role="doc-biblioref">20</a>]</span></td>
</tr>
<tr class="odd">
<td>5</td>
<td>90</td>
<td>WSXM: a software for scanning probe microscopy and a tool for nanotechnology</td>
<td><span class="citation" data-cites="11usgA4JE">[<a href="#ref-11usgA4JE" role="doc-biblioref">21</a>]</span></td>
</tr>
<tr class="even">
<td>0</td>
<td>77</td>
<td>Measuring Distances in Supported Bilayers by Fluorescence Interference-Contrast Microscopy: Polymer Supports and SNARE Proteins</td>
<td><span class="citation" data-cites="JJ2LW0nT">[<a href="#ref-JJ2LW0nT" role="doc-biblioref">22</a>]</span></td>
</tr>
<tr class="odd">
<td>100</td>
<td>99</td>
<td>Toward fluorescence nanoscopy</td>
<td><span class="citation" data-cites="NkJaVLEB">[<a href="#ref-NkJaVLEB" role="doc-biblioref">23</a>]</span></td>
</tr>
<tr class="even">
<td>100</td>
<td>86</td>
<td>In vivo imaging of quantum dots encapsulated in phospholipid micelles</td>
<td><span class="citation" data-cites="18Ot4SCqv">[<a href="#ref-18Ot4SCqv" role="doc-biblioref">24</a>]</span></td>
</tr>
<tr class="odd">
<td>100</td>
<td>99</td>
<td>Water-Soluble Quantum Dots for Multiphoton Fluorescence Imaging in Vivo</td>
<td><span class="citation" data-cites="dYAYQycA">[<a href="#ref-dYAYQycA" role="doc-biblioref">25</a>]</span></td>
</tr>
</tbody>
</table>
<p><strong>Table 1</strong>: Nanotechnology/microscopy papers of interest</p>
<table>
<colgroup>
<col style="width: 25%" />
<col style="width: 25%" />
<col style="width: 25%" />
<col style="width: 25%" />
</colgroup>
<thead>
<tr class="header">
<th>Immunochemistry Percentile</th>
<th>Anatomy Percentile</th>
<th>Title</th>
<th>Reference</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>100</td>
<td>45</td>
<td>Immunoelectron microscopic exploration of the Golgi complex</td>
<td><span class="citation" data-cites="PRPDi07h">[<a href="#ref-PRPDi07h" role="doc-biblioref">26</a>]</span></td>
</tr>
<tr class="even">
<td>100</td>
<td>14</td>
<td>Immunocytochemical and electrophoretic analyses of changes in myosin gene expression in cat posterior temporalis muscle during postnatal development</td>
<td><span class="citation" data-cites="nMCUqTIP">[<a href="#ref-nMCUqTIP" role="doc-biblioref">27</a>]</span></td>
</tr>
<tr class="odd">
<td>98</td>
<td>5</td>
<td>Electron microscopic demonstration of calcitonin in human medullary carcinoma of thyroid by the immuno gold staining method</td>
<td><span class="citation" data-cites="1BGliKdI9">[<a href="#ref-1BGliKdI9" role="doc-biblioref">28</a>]</span></td>
</tr>
<tr class="even">
<td>12</td>
<td>100</td>
<td>Grafting genetically modified cells into the rat brain: characteristics of E. coli β-galactosidase as a reporter gene</td>
<td><span class="citation" data-cites="sc5wklkt">[<a href="#ref-sc5wklkt" role="doc-biblioref">29</a>]</span></td>
</tr>
<tr class="odd">
<td>12</td>
<td>100</td>
<td>Vitamin-D-dependent calcium-binding-protein and parvalbumin occur in bones and teeth</td>
<td><span class="citation" data-cites="TnowewWM">[<a href="#ref-TnowewWM" role="doc-biblioref">30</a>]</span></td>
</tr>
<tr class="even">
<td>3</td>
<td>100</td>
<td>Mapping of brain areas containing RNA homologous to cDNAs encoding the alpha and beta subunits of the rat GABAA gamma-aminobutyrate receptor</td>
<td><span class="citation" data-cites="CBI8FXe4">[<a href="#ref-CBI8FXe4" role="doc-biblioref">31</a>]</span></td>
</tr>
<tr class="odd">
<td>100</td>
<td>100</td>
<td>Studies of the HER-2/neu Proto-Oncogene in Human Breast and Ovarian Cancer</td>
<td><span class="citation" data-cites="jDBckAP">[<a href="#ref-jDBckAP" role="doc-biblioref">32</a>]</span></td>
</tr>
<tr class="even">
<td>100</td>
<td>100</td>
<td>Expression of c-fos Protein in Brain: Metabolic Mapping at the Cellular Level</td>
<td><span class="citation" data-cites="hLSwKASj">[<a href="#ref-hLSwKASj" role="doc-biblioref">33</a>]</span></td>
</tr>
<tr class="odd">
<td>100</td>
<td>100</td>
<td>Proliferating cell nuclear antigen (PCNA) immunolocalization in paraffin sections: An index of cell proliferation with evidence of deregulated expression in some neoplasms</td>
<td><span class="citation" data-cites="hLSwKASj">[<a href="#ref-hLSwKASj" role="doc-biblioref">33</a>]</span></td>
</tr>
</tbody>
</table>
<p><strong>Table 2</strong>: Immunochemistry/anatomy papers of interest</p>
<table>
<colgroup>
<col style="width: 25%" />
<col style="width: 25%" />
<col style="width: 25%" />
<col style="width: 25%" />
</colgroup>
<thead>
<tr class="header">
<th>Proteomics Percentile</th>
<th>Metabolomics Percentile</th>
<th>Title</th>
<th>Reference</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>67</td>
<td>2</td>
<td>Proteomics Standards Initiative: Fifteen Years of Progress and Future Work</td>
<td><span class="citation" data-cites="qVEHb5K">[<a href="#ref-qVEHb5K" role="doc-biblioref">34</a>]</span></td>
</tr>
<tr class="even">
<td>99</td>
<td>0</td>
<td>Limited Environmental Serine and Glycine Confer Brain Metastasis Sensitivity to PHGDH Inhibition</td>
<td><span class="citation" data-cites="MVZ9xwhJ">[<a href="#ref-MVZ9xwhJ" role="doc-biblioref">35</a>]</span></td>
</tr>
<tr class="odd">
<td>100</td>
<td>0</td>
<td>A high-throughput processing service for retention time alignment of complex proteomics and metabolomics LC-MS data</td>
<td><span class="citation" data-cites="197hbKiL0">[<a href="#ref-197hbKiL0" role="doc-biblioref">36</a> ]</span></td>
</tr>
<tr class="even">
<td>0</td>
<td>100</td>
<td>MeltDB: a software platform for the analysis and integration of metabolomics experiment data</td>
<td><span class="citation" data-cites="EvwvRYzE">[<a href="#ref-EvwvRYzE" role="doc-biblioref">37</a>]</span></td>
</tr>
<tr class="odd">
<td>0</td>
<td>98</td>
<td>In silico fragmentation for computer assisted identification of metabolite mass spectra</td>
<td><span class="citation" data-cites="vNOeEVfp">[<a href="#ref-vNOeEVfp" role="doc-biblioref">38</a>]</span></td>
</tr>
<tr class="even">
<td>0</td>
<td>100</td>
<td>The Metabonomic Signature of Celiac Disease</td>
<td><span class="citation" data-cites="UQbCNqUe">[<a href="#ref-UQbCNqUe" role="doc-biblioref">39</a>]</span></td>
</tr>
<tr class="odd">
<td>91</td>
<td>70</td>
<td>Visualization of omics data for systems biology</td>
<td><span class="citation" data-cites="vNOeEVfp">[<a href="#ref-vNOeEVfp" role="doc-biblioref">38</a>]</span></td>
</tr>
<tr class="even">
<td>0</td>
<td>16</td>
<td>FunRich: An open access standalone functional enrichment and interaction network analysis tool</td>
<td><span class="citation" data-cites="vJbGwiV7">[<a href="#ref-vJbGwiV7" role="doc-biblioref">40</a>]</span></td>
</tr>
<tr class="odd">
<td>0</td>
<td>5</td>
<td>Proteomic and Metabolomic Characterization of COVID-19 Patient Sera</td>
<td><span class="citation" data-cites="sc8DMTGM">[<a href="#ref-sc8DMTGM" role="doc-biblioref">41</a>]</span></td>
</tr>
</tbody>
</table>
<p><strong>Table 3</strong>: Proteomics/metabolomics papers of interest</p>
<table>
<colgroup>
<col style="width: 25%" />
<col style="width: 25%" />
<col style="width: 25%" />
<col style="width: 25%" />
</colgroup>
<thead>
<tr class="header">
<th>Computational Biology Percentile</th>
<th>Human Genetics Percentile</th>
<th>Title</th>
<th>Reference</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>99</td>
<td>0</td>
<td>Development of Human Protein Reference Database as an Initial Platform for Approaching Systems Biology in Humans</td>
<td><span class="citation" data-cites="VvZTjIen">[<a href="#ref-VvZTjIen" role="doc-biblioref">42</a>]</span></td>
</tr>
<tr class="even">
<td>100</td>
<td>1</td>
<td>A database for post-genome analysis</td>
<td><span class="citation" data-cites="C4NTSwn4">[<a href="#ref-C4NTSwn4" role="doc-biblioref">43</a>]</span></td>
</tr>
<tr class="odd">
<td>100</td>
<td>1</td>
<td>Use of mass spectrometry-derived data to annotate nucleotide and protein sequence databases</td>
<td><span class="citation" data-cites="G2ZiZdRY">[<a href="#ref-G2ZiZdRY" role="doc-biblioref">44</a>]</span></td>
</tr>
<tr class="even">
<td>12</td>
<td>100</td>
<td>Genetic Discrimination: Perspectives of Consumers</td>
<td><span class="citation" data-cites="mcDRypJj">[<a href="#ref-mcDRypJj" role="doc-biblioref">45</a>]</span></td>
</tr>
<tr class="odd">
<td>0</td>
<td>81</td>
<td>Committee Opinion No. 690: Carrier Screening in the Age of Genomic Medicine</td>
<td><span class="citation" data-cites="PexGnVkD">[<a href="#ref-PexGnVkD" role="doc-biblioref">46</a> ]</span></td>
</tr>
<tr class="even">
<td>23</td>
<td>100</td>
<td>Public health genomics: The end of the beginning</td>
<td><span class="citation" data-cites="Msb0hwgj">[<a href="#ref-Msb0hwgj" role="doc-biblioref">47</a>]</span></td>
</tr>
<tr class="odd">
<td>100</td>
<td>99</td>
<td>Initial sequencing and analysis of the human genome</td>
<td><span class="citation" data-cites="14yhCmDBZ">[<a href="#ref-14yhCmDBZ" role="doc-biblioref">48</a>]</span></td>
</tr>
<tr class="even">
<td>100</td>
<td>100</td>
<td>An STS-Based Map of the Human Genome</td>
<td><span class="citation" data-cites="7ZlRSgYT">[<a href="#ref-7ZlRSgYT" role="doc-biblioref">49</a>]</span></td>
</tr>
<tr class="odd">
<td>100</td>
<td>100</td>
<td>A New Five-Year Plan for the U.S. Human Genome Project</td>
<td><span class="citation" data-cites="153eZYSdV">[<a href="#ref-153eZYSdV" role="doc-biblioref">50</a>]</span></td>
</tr>
</tbody>
</table>
<p><strong>Table 4</strong>: Computational biology/human genetics papers of interest</p>
<h2 id="methods">Methods</h2>
<h4 id="coci">COCI</h4>
<p>We used the March 2022 version of the COCI citation index <span class="citation" data-cites="126JIm8me">[<a href="#ref-126JIm8me" role="doc-biblioref">51</a>]</span> as the source of our citation data.
This dataset contains around 1.3 billion citations from ~73 million bibliographic resources.</p>
<h4 id="selecting-fields">Selecting fields</h4>
<p>To differentiate between scientific fields, we needed a way to map papers to fields.
Fortunately, all the papers in Pubmed Central (https://www.ncbi.nlm.nih.gov/pmc/) have corresponding Medical Subject Headings (MeSH) terms.
While MeSH terms are varied and numerous, the subheadings of the Natural Science Disciplines (H01) category fit our needs.
However, MeSH terms are hierarchical and vary greatly in their size and specificity.
To extract a balanced set of terms, we recursively traversed the tree and selected headings having at least 10000 DOIs without having multiple children that also meet the cutoff.
Our resulting headings contained 45 terms, from “Acoustics” to “Water Microbiology.”</p>
<h4 id="building-single-heading-citation-networks">Building single heading citation networks</h4>
<p>The COCI dataset consists of pairs of Digital Object Identifiers (DOIs).
To change these pairs into a form we could run calculations on, we needed to convert them into networks.
To do so, we created 45 empty networks, one for each previously selected MeSH term.
We then iterated over each pair of DOIs in COCI and added them to a network if the DOIs corresponded to two journal articles written in English, both of which were tagged with the corresponding MeSH heading.</p>
<h4 id="building-combined-networks">Building combined networks</h4>
<p>Because we were interested in the differences between fields, we also needed to build networks from pairs of MeSH headings.
These networks were built via the same process, except that instead of keeping articles corresponding to a single DOI we added a citation to the network if both articles were in the pair of fields, even if the citation occurred across fields.
Running this network-building process yielded 990 two-heading networks.</p>
<h4 id="shuffling-networks">Shuffling networks</h4>
<p>Sampling a graph from the degree distribution while preserving the distribution of degrees in the network was challenging.
Because citation graphs are directed, it is impossible to simply swap pairs of edges and end up with a graph uniformly sampled from the space.
Instead, a more sophisticated three-edge swap method must be used <span class="citation" data-cites="To1WoeSQ">[<a href="#ref-To1WoeSQ" role="doc-biblioref">52</a>]</span>.
Because this algorithm had not been implemented yet in NetworkX <span class="citation" data-cites="15wNo6JtP">[<a href="#ref-15wNo6JtP" role="doc-biblioref">53</a>]</span>, we implemented the code to perform shuffles and submitted our change to the library. <!-- DO YOU WANT TO CITE THE PULL REQUEST HERE VIA URL? -->
With the shuffling code implemented, we created 100 shuffled versions of each of our combined networks to act as a background distribution against which we could compare metrics.</p>
<h4 id="splitting-networks">Splitting networks</h4>
<p>Once we had a collection of shuffled networks, we needed to split them into their constituent fields.
To do so, we reduced the network to solely the nodes that were present in the single heading citation network and kept only citations between these nodes.</p>
<h4 id="running-pagerank">Running PageRank</h4>
<p>We used the NetworkX implementation of PageRank with default parameters to evaluate paper importance within fields.</p>
<h4 id="percentile-score">Percentile score</h4>
<p>To determine the degree to which the papers’ PageRank values were higher or lower than expected, we compared the PageRank values calculated for the true citation networks to the values in the shuffled networks for each paper.
We then recorded the percent of shuffled networks where the paper had a lower PageRank than the true network to derive a single number that described these values.
For example, if a paper had a higher PageRank in the true network than in all the shuffled networks it received a percentile of 100.
Likewise, if it had a lower PageRank in the true network than in all the shuffled networks it received a percentile of 0.</p>
<h4 id="differences-in-percentiles">Differences in percentiles</h4>
<p>A convenient feature of the percentiles was that they were directly comparable between fields.
For manuscripts represented in two fields, the difference in scores was used to estimate its variability in importance.
For example, if a paper had a score of 100 in field A (indicating a higher PageRank in the field than expected given its number of citations and the network structure) and a score of 0 in field B (indicating a lower than expected PageRank), then the large difference in scores indicated the paper was more highly valued in field A than field B.
If the paper had similar scores in both fields, it indicated that the paper was similarly valued in the two fields.</p>
<h4 id="hardwareruntime">Hardware/runtime</h4>
<p>We ran the full analysis pipeline on the RMACC Summit cluster at the University of Colorado.
The pipeline took about a week to run, from downloading the data to analyzing it to visualizing it.
Performance in other contexts will depend heavily on details such as the number of CPU nodes available and the network speed.</p>
<h4 id="server-details">Server details</h4>
<p>Our webserver is built by visualizing our data in Plotly (https://plotly.com/python/plotly-express/) on the Streamlit platform (https://streamlit.io/).
The field pairs made available by the frontend are those with at least 1000 shared papers after filtering out papers with more than a 5% missingness level of their PageRanks after shuffling.
The journals available for visualization are those with at least 25 papers for the given field pair.</p>
<h2 id="discussionconclusion">Discussion/Conclusion</h2>
<p>We analyze hundreds of field-pair citation networks to examine the extent to which article-level importance metrics vary between fields.
As previously reported, we find systematic differences in PageRanks between fields <span class="citation" data-cites="12CYfIcWF hJ4Xxryu">[<a href="#ref-12CYfIcWF" role="doc-biblioref">7</a>,<a href="#ref-hJ4Xxryu" role="doc-biblioref">54</a>]</span> that would warrant some form of normalization when making cross-field comparisons with global statistics.
However, we also find that field-specific differences are not driven solely by differences in citation practices.
Instead, the importance of individual papers appears to differ meaningfully between fields.
Global rankings or efforts to normalize out field-specific effects obscure meaningful differences in manuscript importance between communities.</p>
<p>As with any study, this research has certain limitations.
One example is our selection of MeSH terms to represent fields.
We used MeSH because it is a widely-annotated set of subjects in biomedicine and thresholded MeSH term sizes to balance having enough observations to calculate appropriate statistics with having sufficient granularity to capture fields.
This resulted in fields at the granularity of “biophysics” and “ecology.”
We also have to select a number of swaps to generate a background distribution of PageRanks for each field pair.
We selected three times as many swaps as edges, where each swap modifies three edges, but certain network structures may require a different number.</p>
<p>We also note that there are inherent issues with assigning manuscript importance scores.
We sought to understand the extent to which such rankings were stable between fields after correcting for field-specific citation practices.
We found limited stability between fields, mostly between closely-related fields, suggesting that the concept of a universal ranking of importances is difficult to justify.
In the way that reducing a distribution to a Journal Impact Factor distorts assessment, attempting to use a single universal score to represent importance across fields poses similar challenges at the level of individual manucripts.
Furthermore, this work’s natural progression would extend to estimating the importance of individual manuscripts to individual researchers.
Thus, a holistic measure of importance would need to include a distribution of scores not only across fields but across researchers.
It may ultimately be impossible to calculate a meaningful importance score.
The lack of ground truth for importance is an inherent feature, not a bug, of science’s step-wide progression.</p>
<p>Shifting from the perspective of evaluation to discovery can reveal more appropriate uses for these types of statistics.
Field-pair calculations for such metrics may help with self-directed learning of new fields.
An expert in one field, e.g., computational biology, who aims to learn more about genetics may find manuscripts with high importance in genetics and low importance in computational biology to be important reads.
These represent manuscripts not currently widely cited in one’s field but highly influential in a target field.
Our application can reveal these manuscripts for MeSH field pairs, and our source code allows others to perform our analysis with different granularity.</p>
<h3 id="code-and-data-availability">Code and Data Availability</h3>
<p>The code to reproduce this work can be found at https://github.com/greenelab/indices.<br />
The data used for this project can be downloaded from Zenodo at TODO.
Our work meets the bronze standard of reproducibility <span class="citation" data-cites="PETW01rJ">[<a href="#ref-PETW01rJ" role="doc-biblioref">55</a>]</span> and fulfills aspects of the silver and gold standards including deterministic operation.</p>
<h3 id="acknowledgements">Acknowledgements</h3>
<p>We would like to thank Jake Crawford for reviewing code that went into this project.<br />
We would also like to thank the past and present members of GreeneLab who gave feedback on this project during lab meetings.<br />
This work utilized resources from the University of Colorado Boulder Research Computing Group, which is supported by the National Science Foundation (awards ACI-1532235 and ACI-1532236).</p>
<h4 id="funding">Funding</h4>
<p>This work was supported by grants from the National Institutes of Health’s National Human Genome Research Institute (NHGRI) under award R01 HG010067 and the Gordon and Betty Moore Foundation (GBMF 4552) to CSG.
The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p>
<h2 class="page_break_before" id="references">References</h2>
<!-- Explicitly insert bibliography here -->
<div id="refs" class="references csl-bib-body" role="doc-bibliography">
<div id="ref-qXsETMbA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1. </div><div class="csl-right-inline"><strong>An index to quantify an individual's scientific research output</strong> <div class="csl-block">JE Hirsch</div> <em>Proceedings of the National Academy of Sciences</em> (2005-11-07) <a href="https://doi.org/cbq6dz">https://doi.org/cbq6dz</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1073/pnas.0507655102">10.1073/pnas.0507655102</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16275915">16275915</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1283832">PMC1283832</a></div></div>
</div>
<div id="ref-16V0td5f2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2. </div><div class="csl-right-inline"><strong>New Tools for Improving and Evaluating The Effectiveness of Research</strong> <div class="csl-block">Irving H Sher, Eugene Garfield</div> <em>Research Program Effectiveness</em> (1965-06-27)</div>
</div>
<div id="ref-zeuTsDVX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">3. </div><div class="csl-right-inline"><strong>The PageRank Citation Ranking: Bringing Order to the Web.</strong> <div class="csl-block">Lawrence Page, Sergey Brin, Rajeev Motwani, Terry Winograd</div> <em>Stanford InfoLab</em> (1999)</div>
</div>
<div id="ref-UTLLCTKy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">4. </div><div class="csl-right-inline"><strong>Large teams develop and small teams disrupt science and technology</strong> <div class="csl-block">Lingfei Wu, Dashun Wang, James A Evans</div> <em>Nature</em> (2019-02) <a href="https://doi.org/gfvnb9">https://doi.org/gfvnb9</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41586-019-0941-9">10.1038/s41586-019-0941-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30760923">30760923</a></div></div>
</div>
<div id="ref-kZI40ZXS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">5. </div><div class="csl-right-inline"><strong>Relative Citation Ratio (RCR): A New Metric That Uses Citation Rates to Measure Influence at the Article Level</strong> <div class="csl-block">BIan Hutchins, Xin Yuan, James M Anderson, George M Santangelo</div> <em>PLOS Biology</em> (2016-09-06) <a href="https://doi.org/f88zk2">https://doi.org/f88zk2</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1371/journal.pbio.1002541">10.1371/journal.pbio.1002541</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27599104">27599104</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5012559">PMC5012559</a></div></div>
</div>
<div id="ref-11IkPxdIL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">6. </div><div class="csl-right-inline"><strong>Measuring contextual citation impact of scientific journals</strong> <div class="csl-block">Henk F Moed</div> <em>Journal of Informetrics</em> (2010-07) <a href="https://doi.org/dpbgj9">https://doi.org/dpbgj9</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.joi.2010.01.002">10.1016/j.joi.2010.01.002</a></div></div>
</div>
<div id="ref-12CYfIcWF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">7. </div><div class="csl-right-inline"><strong>Collective topical PageRank: a model to evaluate the topic-dependent academic impact of scientific papers</strong> <div class="csl-block">Yongjun Zhang, Jialin Ma, Zijian Wang, Bolun Chen, Yongtao Yu</div> <em>Scientometrics</em> (2017-12-23) <a href="https://doi.org/gc4b2s">https://doi.org/gc4b2s</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/s11192-017-2626-1">10.1007/s11192-017-2626-1</a></div></div>
</div>
<div id="ref-PGRGcmLi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">8. </div><div class="csl-right-inline"><strong>Are disruption index indicators convergently valid? The comparison of several indicator variants with assessments by peers</strong> <div class="csl-block">Lutz Bornmann, Sitaram Devarakonda, Alexander Tekles, George Chacko</div> <em>Quantitative Science Studies</em> (2020-08) <a href="https://doi.org/gq2ts5">https://doi.org/gq2ts5</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1162/qss_a_00068">10.1162/qss_a_00068</a></div></div>
</div>
<div id="ref-62Q1rmv4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">9. </div><div class="csl-right-inline"><strong>Impact Factor Distortions</strong> <div class="csl-block">Bruce Alberts</div> <em>Science</em> (2013-05-17) <a href="https://doi.org/mjm">https://doi.org/mjm</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1126/science.1240319">10.1126/science.1240319</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23687012">23687012</a></div></div>
</div>
<div id="ref-Hw4ZDEnz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">10. </div><div class="csl-right-inline"><strong>Citation patterns in economics and beyond</strong> <div class="csl-block">Matthias Aistleitner, Jakob Kapeller, Stefan Steinerberger</div> <em>Science in Context</em> (2019-12) <a href="https://doi.org/gq62s8">https://doi.org/gq62s8</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1017/s0269889720000022">10.1017/s0269889720000022</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32202238">32202238</a></div></div>
</div>
<div id="ref-7n76ZZPt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">11. </div><div class="csl-right-inline"><strong>Disruptive papers published in Scientometrics: meaningful results by using an improved variant of the disruption index originally proposed by Wu, Wang, and Evans (2019)</strong> <div class="csl-block">Lutz Bornmann, Sitaram Devarakonda, Alexander Tekles, George Chacko</div> <em>Scientometrics</em> (2020-03-14) <a href="https://doi.org/ggzzxd">https://doi.org/ggzzxd</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/s11192-020-03406-8">10.1007/s11192-020-03406-8</a></div></div>
</div>
<div id="ref-3FJspt0W" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">12. </div><div class="csl-right-inline"><strong>Quantifying and suppressing ranking bias in a large citation network</strong> <div class="csl-block">Giacomo Vaccario, Matúš Medo, Nicolas Wider, Manuel Sebastian Mariani</div> <em>Journal of Informetrics</em> (2017-08) <a href="https://doi.org/gbzjdh">https://doi.org/gbzjdh</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.joi.2017.05.014">10.1016/j.joi.2017.05.014</a></div></div>
</div>
<div id="ref-15JxaxOiR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">13. </div><div class="csl-right-inline"><strong>Quantitative evaluation of alternative field normalization procedures</strong> <div class="csl-block">Yunrong Li, Filippo Radicchi, Claudio Castellano, Javier Ruiz-Castillo</div> <em>Journal of Informetrics</em> (2013-07) <a href="https://doi.org/f48tvt">https://doi.org/f48tvt</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.joi.2013.06.001">10.1016/j.joi.2013.06.001</a></div></div>
</div>
<div id="ref-AkQLg7kt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">14. </div><div class="csl-right-inline"><strong>When a journal is both at the ‘top’ and the ‘bottom’: the illogicality of conflating citation-based metrics with quality</strong> <div class="csl-block">Shannon Mason, Lenandlar Singh</div> <em>Scientometrics</em> (2022-05-25) <a href="https://doi.org/gq2468">https://doi.org/gq2468</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/s11192-022-04402-w">10.1007/s11192-022-04402-w</a></div></div>
</div>
<div id="ref-4TkOo7Oy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">15. </div><div class="csl-right-inline"><strong>Eigenfactor: Measuring the value and prestige of scholarly journals</strong> <div class="csl-block">Carl Bergstrom</div> <em>College &amp;amp; Research Libraries News</em> (2007-05-01) <a href="https://doi.org/gf24tg">https://doi.org/gf24tg</a> <div class="csl-block">DOI: <a href="https://doi.org/10.5860/crln.68.5.7804">10.5860/crln.68.5.7804</a></div></div>
</div>
<div id="ref-ErD8Jjn8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">16. </div><div class="csl-right-inline"><strong>Cited references and Medical Subject Headings (MeSH) as two different knowledge representations: clustering and mappings at the paper level</strong> <div class="csl-block">Loet Leydesdorff, Jordan A Comins, Aaron A Sorensen, Lutz Bornmann, Iina Hellsten</div> <em>Scientometrics</em> (2016-10-08) <a href="https://doi.org/gc8zk4">https://doi.org/gc8zk4</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/s11192-016-2119-7">10.1007/s11192-016-2119-7</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27942085">27942085</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5124055">PMC5124055</a></div></div>
</div>
<div id="ref-1GphepDNl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">17. </div><div class="csl-right-inline"><strong>A robust DNA mechanical device controlled by hybridization topology</strong> <div class="csl-block">Hao Yan, Xiaoping Zhang, Zhiyong Shen, Nadrian C Seeman</div> <em>Nature</em> (2002-01) <a href="https://doi.org/czh8hg">https://doi.org/czh8hg</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/415062a">10.1038/415062a</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11780115">11780115</a></div></div>
</div>
<div id="ref-NmOwHa9H" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">18. </div><div class="csl-right-inline"><strong>Bioadhesive poly(methyl methacrylate) microdevices for controlled drug delivery</strong> <div class="csl-block">Sarah L Tao, Michael W Lubeley, Tejal A Desai</div> <em>Journal of Controlled Release</em> (2003-03) <a href="https://doi.org/c7fpg4">https://doi.org/c7fpg4</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/s0168-3659(03)00005-1">10.1016/s0168-3659(03)00005-1</a></div></div>
</div>
<div id="ref-IkV2K836" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">19. </div><div class="csl-right-inline"><strong>DNA-Templated Self-Assembly of Protein Arrays and Highly Conductive Nanowires</strong> <div class="csl-block">Hao Yan, Sung Ha Park, Gleb Finkelstein, John H Reif, Thomas H LaBean</div> <em>Science</em> (2003-09-26) <a href="https://doi.org/bfgvgf">https://doi.org/bfgvgf</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1126/science.1089389">10.1126/science.1089389</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14512621">14512621</a></div></div>
</div>
<div id="ref-19ryujP9j" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">20. </div><div class="csl-right-inline"><strong>Photostable Luminescent Nanoparticles as Biological Label for Cell Recognition of System Lupus Erythematosus Patients</strong> <div class="csl-block">Xiaoxiao He, Kemin Wang, Weihong Tan, Jun Li, Xiaohai Yang, Shasheng Huang, Dan Xiao</div> <em>Journal of Nanoscience and Nanotechnology</em> (2002-07-01) <a href="https://doi.org/dcj5cg">https://doi.org/dcj5cg</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1166/jnn.2002.105">10.1166/jnn.2002.105</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12908257">12908257</a></div></div>
</div>
<div id="ref-11usgA4JE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">21. </div><div class="csl-right-inline"><strong>WSXM: A software for scanning probe microscopy and a tool for nanotechnology</strong> <div class="csl-block">I Horcas, R Fernández, JM Gómez-Rodríguez, J Colchero, J Gómez-Herrero, AM Baro</div> <em>Review of Scientific Instruments</em> (2007-01) <div class="csl-block">DOI: <a href="https://doi.org/10.1063/1.2432410">10.1063/1.2432410</a></div></div>
</div>
<div id="ref-JJ2LW0nT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">22. </div><div class="csl-right-inline"><strong>Measuring Distances in Supported Bilayers by Fluorescence Interference-Contrast Microscopy: Polymer Supports and SNARE Proteins</strong> <div class="csl-block">Volker Kiessling, Lukas K Tamm</div> <em>Biophysical Journal</em> (2003-01) <a href="https://doi.org/dqsg2c">https://doi.org/dqsg2c</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/s0006-3495(03)74861-9">10.1016/s0006-3495(03)74861-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12524294">12524294</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1302622">PMC1302622</a></div></div>
</div>
<div id="ref-NkJaVLEB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">23. </div><div class="csl-right-inline"><strong>Toward fluorescence nanoscopy</strong> <div class="csl-block">Stefan W Hell</div> <em>Nature Biotechnology</em> (2003-10-31) <a href="https://doi.org/dnzt3b">https://doi.org/dnzt3b</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nbt895">10.1038/nbt895</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14595362">14595362</a></div></div>
</div>
<div id="ref-18Ot4SCqv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">24. </div><div class="csl-right-inline"><strong>In Vivo Imaging of Quantum Dots Encapsulated in Phospholipid Micelles</strong> <div class="csl-block">Benoit Dubertret, Paris Skourides, David J Norris, Vincent Noireaux, Ali H Brivanlou, Albert Libchaber</div> <em>Science</em> (2002-11-29) <a href="https://doi.org/dd6sqp">https://doi.org/dd6sqp</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1126/science.1077194">10.1126/science.1077194</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12459582">12459582</a></div></div>
</div>
<div id="ref-dYAYQycA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">25. </div><div class="csl-right-inline"><strong>Water-Soluble Quantum Dots for Multiphoton Fluorescence Imaging in Vivo</strong> <div class="csl-block">Daniel R Larson, Warren R Zipfel, Rebecca M Williams, Stephen W Clark, Marcel P Bruchez, Frank W Wise, Watt W Webb</div> <em>Science</em> (2003-05-30) <a href="https://doi.org/cn9j76">https://doi.org/cn9j76</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1126/science.1083780">10.1126/science.1083780</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12775841">12775841</a></div></div>
</div>
<div id="ref-PRPDi07h" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">26. </div><div class="csl-right-inline"><strong>Immunoelectron microscopic exploration of the Golgi complex.</strong> <div class="csl-block">JW Slot, HJ Geuze</div> <em>Journal of Histochemistry &amp;amp; Cytochemistry</em> (1983-08) <a href="https://doi.org/dxxzxg">https://doi.org/dxxzxg</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1177/31.8.6863900">10.1177/31.8.6863900</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/6863900">6863900</a></div></div>
</div>
<div id="ref-nMCUqTIP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">27. </div><div class="csl-right-inline"><strong>Immunocytochemical and electrophoretic analyses of changes in myosin gene expression in cat posterior temporalis muscle during postnatal development</strong> <div class="csl-block">JFY Hoh, S Hughes, C Chow, PT Hale, RB Fitzsimons</div> <em>Journal of Muscle Research and Cell Motility</em> (1988-02) <a href="https://doi.org/d72278">https://doi.org/d72278</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/bf01682147">10.1007/bf01682147</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/3392187">3392187</a></div></div>
</div>
<div id="ref-1BGliKdI9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">28. </div><div class="csl-right-inline"><strong>Electron microscopic demonstration of calcitonin in human medullary carcinoma of thyroid by the immuno gold staining method</strong> <div class="csl-block">J Dämmrich, W Ormanns, R Schäffer</div> <em>Histochemistry</em> (1984) <a href="https://doi.org/ct253c">https://doi.org/ct253c</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/bf00514331">10.1007/bf00514331</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/6511490">6511490</a></div></div>
</div>
<div id="ref-sc5wklkt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">29. </div><div class="csl-right-inline"><strong>Grafting genetically modified cells into the rat brain: characteristics of E. coli β-galactosidase as a reporter gene</strong> <div class="csl-block">S Shimohama, MB Rosenberg, AM Fagan, JA Wolff, MP Short, XO Breakefield, T Friedmann, FH Gage</div> <em>Molecular Brain Research</em> (1989-06) <a href="https://doi.org/dptnbm">https://doi.org/dptnbm</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/0169-328x(89)90061-2">10.1016/0169-328x(89)90061-2</a></div></div>
</div>
<div id="ref-TnowewWM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">30. </div><div class="csl-right-inline"><strong>Vitamin-D-dependent calcium-binding-protein and parvalbumin occur in bones and teeth</strong> <div class="csl-block">MR Celio, AW Norman, CW Heizmann</div> <em>Calcified Tissue International</em> (1984-12) <a href="https://doi.org/fdnfdg">https://doi.org/fdnfdg</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/bf02405306">10.1007/bf02405306</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/6423230">6423230</a></div></div>
</div>
<div id="ref-CBI8FXe4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">31. </div><div class="csl-right-inline"><strong>Mapping of brain areas containing RNA homologous to cDNAs encoding the alpha and beta subunits of the rat GABAA gamma-aminobutyrate receptor.</strong> <div class="csl-block">JM Séquier, JG Richards, P Malherbe, GW Price, S Mathews, H Möhler</div> <em>Proceedings of the National Academy of Sciences</em> (1988-10) <a href="https://doi.org/fv2p49">https://doi.org/fv2p49</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1073/pnas.85.20.7815">10.1073/pnas.85.20.7815</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/2845424">2845424</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC282284">PMC282284</a></div></div>
</div>
<div id="ref-jDBckAP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">32. </div><div class="csl-right-inline"><strong>Studies of the HER-2/
&lt;i&gt;neu&lt;/i&gt;
Proto-Oncogene in Human Breast and Ovarian Cancer</strong> <div class="csl-block">Dennis J Slamon, William Godolphin, Lovell A Jones, John A Holt, Steven G Wong, Duane E Keith, Wendy J Levin, Susan G Stuart, Judy Udove, Axel Ullrich, Michael F Press</div> <em>Science</em> (1989-05-12) <a href="https://doi.org/cngtqx">https://doi.org/cngtqx</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1126/science.2470152">10.1126/science.2470152</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/2470152">2470152</a></div></div>
</div>
<div id="ref-hLSwKASj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">33. </div><div class="csl-right-inline"><strong>Expression of c-
&lt;i&gt;fos&lt;/i&gt;
Protein in Brain: Metabolic Mapping at the Cellular Level</strong> <div class="csl-block">SM Sagar, FR Sharp, T Curran</div> <em>Science</em> (1988-06-03) <a href="https://doi.org/b39h2t">https://doi.org/b39h2t</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1126/science.3131879">10.1126/science.3131879</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/3131879">3131879</a></div></div>
</div>
<div id="ref-qVEHb5K" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">34. </div><div class="csl-right-inline"><strong>Proteomics Standards Initiative: Fifteen Years of Progress and Future Work</strong> <div class="csl-block">Eric W Deutsch, Sandra Orchard, Pierre-Alain Binz, Wout Bittremieux, Martin Eisenacher, Henning Hermjakob, Shin Kawano, Henry Lam, Gerhard Mayer, Gerben Menschaert, … Andrew R Jones</div> <em>Journal of Proteome Research</em> (2017-09-15) <a href="https://doi.org/gbw99d">https://doi.org/gbw99d</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.7b00370">10.1021/acs.jproteome.7b00370</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28849660">28849660</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5715286">PMC5715286</a></div></div>
</div>
<div id="ref-MVZ9xwhJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">35. </div><div class="csl-right-inline"><strong>Limited Environmental Serine and Glycine Confer Brain Metastasis Sensitivity to PHGDH Inhibition</strong> <div class="csl-block">Bryan Ngo, Eugenie Kim, Victoria Osorio-Vasquez, Sophia Doll, Sophia Bustraan, Roger J Liang, Alba Luengo, Shawn M Davidson, Ahmed Ali, Gino B Ferraro, … Michael E Pacold</div> <em>Cancer Discovery</em> (2020-09-01) <a href="https://doi.org/ghf85j">https://doi.org/ghf85j</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1158/2159-8290.cd-19-1228">10.1158/2159-8290.cd-19-1228</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32571778">32571778</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7483776">PMC7483776</a></div></div>
</div>
<div id="ref-197hbKiL0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">36. </div><div class="csl-right-inline"><strong>A high-throughput processing service for retention time alignment of complex proteomics and metabolomics LC-MS data</strong> <div class="csl-block">Isthiaq Ahmad, Frank Suits, Berend Hoekman, Morris A Swertz, Heorhiy Byelas, Martijn Dijkstra, Rob Hooft, Dmitry Katsubo, Bas van Breukelen, Rainer Bischoff, Peter Horvatovich</div> <em>Bioinformatics</em> (2011-02-23) <a href="https://doi.org/cxsszv">https://doi.org/cxsszv</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/bioinformatics/btr094">10.1093/bioinformatics/btr094</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21349866">21349866</a></div></div>
</div>
<div id="ref-EvwvRYzE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">37. </div><div class="csl-right-inline"><strong>MeltDB: a software platform for the analysis and integration of metabolomics experiment data</strong> <div class="csl-block">Heiko Neuweger, Stefan P Albaum, Michael Dondrup, Marcus Persicke, Tony Watt, Karsten Niehaus, Jens Stoye, Alexander Goesmann</div> <em>Bioinformatics</em> (2008-09-02) <a href="https://doi.org/fds6vt">https://doi.org/fds6vt</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/bioinformatics/btn452">10.1093/bioinformatics/btn452</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18765459">18765459</a></div></div>
</div>
<div id="ref-vNOeEVfp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">38. </div><div class="csl-right-inline"><strong>In silico fragmentation for computer assisted identification of metabolite mass spectra</strong> <div class="csl-block">Sebastian Wolf, Stephan Schmidt, Matthias Müller-Hannemann, Steffen Neumann</div> <em>BMC Bioinformatics</em> (2010-03-22) <a href="https://doi.org/d7gpf5">https://doi.org/d7gpf5</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1186/1471-2105-11-148">10.1186/1471-2105-11-148</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20307295">20307295</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2853470">PMC2853470</a></div></div>
</div>
<div id="ref-UQbCNqUe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">39. </div><div class="csl-right-inline"><strong>The Metabonomic Signature of Celiac Disease</strong> <div class="csl-block">Ivano Bertini, Antonio Calabrò, Valeria De Carli, Claudio Luchinat, Stefano Nepi, Berardino Porfirio, Daniela Renzi, Edoardo Saccenti, Leonardo Tenori</div> <em>Journal of Proteome Research</em> (2008-12-11) <a href="https://doi.org/c6sdnp">https://doi.org/c6sdnp</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr800548z">10.1021/pr800548z</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19072164">19072164</a></div></div>
</div>
<div id="ref-vJbGwiV7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">40. </div><div class="csl-right-inline"><strong>FunRich: An open access standalone functional enrichment and interaction network analysis tool</strong> <div class="csl-block">Mohashin Pathan, Shivakumar Keerthikumar, Ching-Seng Ang, Lahiru Gangoda, Camelia YJ Quek, Nicholas A Williamson, Dmitri Mouradov, Oliver M Sieber, Richard J Simpson, Agus Salim, … Suresh Mathivanan</div> <em>PROTEOMICS</em> (2015-06-17) <a href="https://doi.org/f278rp">https://doi.org/f278rp</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/pmic.201400515">10.1002/pmic.201400515</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25921073">25921073</a></div></div>
</div>
<div id="ref-sc8DMTGM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">41. </div><div class="csl-right-inline"><strong>Proteomic and Metabolomic Characterization of COVID-19 Patient Sera</strong> <div class="csl-block">Bo Shen, Xiao Yi, Yaoting Sun, Xiaojie Bi, Juping Du, Chao Zhang, Sheng Quan, Fangfei Zhang, Rui Sun, Liujia Qian, … Tiannan Guo</div> <em>Cell</em> (2020-07) <a href="https://doi.org/gg2cck">https://doi.org/gg2cck</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.cell.2020.05.032">10.1016/j.cell.2020.05.032</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32492406">32492406</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7254001">PMC7254001</a></div></div>
</div>
<div id="ref-VvZTjIen" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">42. </div><div class="csl-right-inline"><strong>Development of Human Protein Reference Database as an Initial Platform for Approaching Systems Biology in Humans</strong> <div class="csl-block">Suraj Peri, JDaniel Navarro, Ramars Amanchy, Troels Z Kristiansen, Chandra Kiran Jonnalagadda, Vineeth Surendranath, Vidya Niranjan, Babylakshmi Muthusamy, TKB Gandhi, Mads Gronborg, … Akhilesh Pandey</div> <em>Genome Research</em> (2003-10) <a href="https://doi.org/bc8cnv">https://doi.org/bc8cnv</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/gr.1680803">10.1101/gr.1680803</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14525934">14525934</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC403728">PMC403728</a></div></div>
</div>
<div id="ref-C4NTSwn4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">43. </div><div class="csl-right-inline"><strong>A database for post-genome analysis</strong> <div class="csl-block">Minoru Kanehisa</div> <em>Trends in Genetics</em> (1997-09) <a href="https://doi.org/cfgb98">https://doi.org/cfgb98</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/s0168-9525(97)01223-7">10.1016/s0168-9525(97)01223-7</a></div></div>
</div>
<div id="ref-G2ZiZdRY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">44. </div><div class="csl-right-inline"><strong>Use of mass spectrometry-derived data to annotate nucleotide and protein sequence databases</strong> <div class="csl-block">Matthias Mann, Akhilesh Pandey</div> <em>Trends in Biochemical Sciences</em> (2001-01) <a href="https://doi.org/ch565r">https://doi.org/ch565r</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/s0968-0004(00)01726-6">10.1016/s0968-0004(00)01726-6</a></div></div>
</div>
<div id="ref-mcDRypJj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">45. </div><div class="csl-right-inline"><strong>Genetic Discrimination: Perspectives of Consumers</strong> <div class="csl-block">EVirginia Lapham, Chahira Kozma, Joan O Weiss</div> <em>Science</em> (1996-10-25) <a href="https://doi.org/df7k88">https://doi.org/df7k88</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1126/science.274.5287.621">10.1126/science.274.5287.621</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/8849455">8849455</a></div></div>
</div>
<div id="ref-PexGnVkD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">46. </div><div class="csl-right-inline"><strong>Committee Opinion No. 690: Carrier Screening in the Age of Genomic Medicine</strong> <div class="csl-block">Obstetrics &amp;amp; Gynecology</div> <em>Ovid Technologies (Wolters Kluwer Health)</em> (2017-03) <a href="https://doi.org/f92g56">https://doi.org/f92g56</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1097/aog.0000000000001951">10.1097/aog.0000000000001951</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28225425">28225425</a></div></div>
</div>
<div id="ref-Msb0hwgj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">47. </div><div class="csl-right-inline"><strong>Public health genomics: The end of the beginning</strong> <div class="csl-block">Muin J Khoury</div> <em>Genetics in Medicine</em> (2011-03) <a href="https://doi.org/bjsxzk">https://doi.org/bjsxzk</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1097/gim.0b013e31821024ca">10.1097/gim.0b013e31821024ca</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21311338">21311338</a></div></div>
</div>
<div id="ref-14yhCmDBZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">48. </div><div class="csl-right-inline"><strong>Initial sequencing and analysis of the human genome</strong><div class="csl-block">, Eric S Lander, Lauren M Linton, Bruce Birren, Chad Nusbaum, Michael C Zody, Jennifer Baldwin, Keri Devon, Ken Dewar, Michael Doyle, … </div> <em>Nature</em> (2001-02-15) <a href="https://doi.org/bfpgjh">https://doi.org/bfpgjh</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/35057062">10.1038/35057062</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11237011">11237011</a></div></div>
</div>
<div id="ref-7ZlRSgYT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">49. </div><div class="csl-right-inline"><strong>An STS-Based Map of the Human Genome</strong> <div class="csl-block">Thomas J Hudson, Lincoln D Stein, Sebastian S Gerety, Junli Ma, Andrew B Castle, James Silva, Donna K Slonim, Rafael Baptista, Leonid Kruglyak, Shu-Hua Xu, … Eric S Lander</div> <em>Science</em> (1995-12-22) <a href="https://doi.org/ftf">https://doi.org/ftf</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1126/science.270.5244.1945">10.1126/science.270.5244.1945</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/8533086">8533086</a></div></div>
</div>
<div id="ref-153eZYSdV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">50. </div><div class="csl-right-inline"><strong>A New Five-Year Plan for the U.S. Human Genome Project</strong> <div class="csl-block">Francis Collins, David Galas</div> <em>Science</em> (1993-10) <a href="https://doi.org/fwkrnb">https://doi.org/fwkrnb</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1126/science.8211127">10.1126/science.8211127</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/8211127">8211127</a></div></div>
</div>
<div id="ref-126JIm8me" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">51. </div><div class="csl-right-inline"><strong>Software review: COCI, the OpenCitations Index of Crossref open DOI-to-DOI citations</strong> <div class="csl-block">Ivan Heibi, Silvio Peroni, David Shotton</div> <em>Scientometrics</em> (2019-09-14) <a href="https://doi.org/ggzz8b">https://doi.org/ggzz8b</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/s11192-019-03217-6">10.1007/s11192-019-03217-6</a></div></div>
</div>
<div id="ref-To1WoeSQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">52. </div><div class="csl-right-inline"><strong>A simple Havel-Hakimi type algorithm to realize graphical degree sequences of directed graphs</strong> <div class="csl-block">Péter L Erdős, István Miklós, Zoltán Toroczkai</div> <em>arXiv</em> (2010-01-21) <a href="https://arxiv.org/abs/0905.4913">https://arxiv.org/abs/0905.4913</a></div>
</div>
<div id="ref-15wNo6JtP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">53. </div><div class="csl-right-inline"><strong>Exploring Network Structure, Dynamics, and Function using NetworkX</strong> <div class="csl-block">Aric A Hagberg, Daniel A Schult, Pieter J Swart</div> <em>Proceedings of the 7th Python in Science conference</em> (2008)</div>
</div>
<div id="ref-hJ4Xxryu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">54. </div><div class="csl-right-inline"><strong>Topic-based Pagerank: toward a topic-level scientific evaluation</strong> <div class="csl-block">Erjia Yan</div> <em>Scientometrics</em> (2014-05-06) <a href="https://doi.org/f6br99">https://doi.org/f6br99</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/s11192-014-1308-5">10.1007/s11192-014-1308-5</a></div></div>
</div>
<div id="ref-PETW01rJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">55. </div><div class="csl-right-inline"><strong>Reproducibility standards for machine learning in the life sciences</strong> <div class="csl-block">Benjamin J Heil, Michael M Hoffman, Florian Markowetz, Su-In Lee, Casey S Greene, Stephanie C Hicks</div> <em>Nature Methods</em> (2021-08-30) <a href="https://doi.org/gmnnqh">https://doi.org/gmnnqh</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41592-021-01256-7">10.1038/s41592-021-01256-7</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34462593">34462593</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9131851">PMC9131851</a></div></div>
</div>
</div>
<!-- default theme -->

<style>
  /* import google fonts */
  @import url("https://fonts.googleapis.com/css?family=Open+Sans:400,600,700");
  @import url("https://fonts.googleapis.com/css?family=Source+Code+Pro");

  /* -------------------------------------------------- */
  /* global */
  /* -------------------------------------------------- */

  /* all elements */
  * {
    /* force sans-serif font unless specified otherwise */
    font-family: "Open Sans", "Helvetica", sans-serif;

    /* prevent text inflation on some mobile browsers */
    -webkit-text-size-adjust: none !important;
    -moz-text-size-adjust: none !important;
    -o-text-size-adjust: none !important;
    text-size-adjust: none !important;
  }

  @media only screen {
    /* "page" element */
    body {
      position: relative;
      box-sizing: border-box;
      font-size: 12pt;
      line-height: 1.5;
      max-width: 8.5in;
      margin: 20px auto;
      padding: 40px;
      border-radius: 5px;
      border: solid 1px #bdbdbd;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      background: #ffffff;
    }
  }

  /* when on screen < 8.5in wide */
  @media only screen and (max-width: 8.5in) {
    /* "page" element */
    body {
      padding: 20px;
      margin: 0;
      border-radius: 0;
      border: none;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05) inset;
      background: none;
    }
  }

  /* -------------------------------------------------- */
  /* headings */
  /* -------------------------------------------------- */

  /* all headings */
  h1,
  h2,
  h3,
  h4,
  h5,
  h6 {
    margin: 20px 0;
    padding: 0;
    font-weight: bold;
  }

  /* biggest heading */
  h1 {
    margin: 40px 0;
    text-align: center;
  }

  /* second biggest heading */
  h2 {
    margin-top: 30px;
    padding-bottom: 5px;
    border-bottom: solid 1px #bdbdbd;
  }

  /* heading font sizes */
  h1 {
    font-size: 2em;
  }
  h2 {
    font-size: 1.5em;
  }
  h3 {
    font-size: 1.35em;
  }
  h4 {
    font-size: 1.25em;
  }
  h5 {
    font-size: 1.15em;
  }
  h6 {
    font-size: 1em;
  }

  /* -------------------------------------------------- */
  /* manuscript header */
  /* -------------------------------------------------- */

  /* manuscript title */
  header > h1 {
    margin: 0;
  }

  /* manuscript title caption text (ie "automatically generated on") */
  header + p {
    text-align: center;
    margin-top: 10px;
  }

  /* -------------------------------------------------- */
  /* text elements */
  /* -------------------------------------------------- */

  /* links */
  a {
    color: #2196f3;
    overflow-wrap: break-word;
  }

  /* superscripts and subscripts */
  sub,
  sup {
    /* prevent from affecting line height */
    line-height: 0;
  }

  /* unordered and ordered lists*/
  ul,
  ol {
    padding-left: 20px;
  }

  /* class for styling text semibold */
  .semibold {
    font-weight: 600;
  }

  /* class for styling elements horizontally left aligned */
  .left {
    display: block;
    text-align: left;
    margin-left: auto;
    margin-right: 0;
    justify-content: left;
  }

  /* class for styling elements horizontally centered */
  .center {
    display: block;
    text-align: center;
    margin-left: auto;
    margin-right: auto;
    justify-content: center;
  }

  /* class for styling elements horizontally right aligned */
  .right {
    display: block;
    text-align: right;
    margin-left: 0;
    margin-right: auto;
    justify-content: right;
  }

  /* -------------------------------------------------- */
  /* section elements */
  /* -------------------------------------------------- */

  /* horizontal divider line */
  hr {
    border: none;
    height: 1px;
    background: #bdbdbd;
  }

  /* paragraphs, horizontal dividers, figures, tables, code */
  p,
  hr,
  figure,
  table,
  pre {
    /* treat all as "paragraphs", with consistent vertical margins */
    margin-top: 20px;
    margin-bottom: 20px;
  }

  /* -------------------------------------------------- */
  /* figures */
  /* -------------------------------------------------- */

  /* figure */
  figure {
    max-width: 100%;
    margin-left: auto;
    margin-right: auto;
  }

  /* figure caption */
  figcaption {
    padding: 0;
    padding-top: 10px;
  }

  /* figure image element */
  figure > img,
  figure > svg {
    max-width: 100%;
    display: block;
    margin-left: auto;
    margin-right: auto;
  }

  /* figure auto-number */
  img + figcaption > span:first-of-type,
  svg + figcaption > span:first-of-type {
    font-weight: bold;
    margin-right: 5px;
  }

  /* -------------------------------------------------- */
  /* tables */
  /* -------------------------------------------------- */

  /* table */
  table {
    border-collapse: collapse;
    border-spacing: 0;
    width: 100%;
    margin-left: auto;
    margin-right: auto;
  }

  /* table cells */
  th,
  td {
    border: solid 1px #bdbdbd;
    padding: 10px;
    /* squash table if too wide for page by forcing line breaks */
    overflow-wrap: break-word;
    word-break: break-word;
  }

  /* header row and even rows */
  th,
  tr:nth-child(2n) {
    background-color: #fafafa;
  }

  /* odd rows */
  tr:nth-child(2n + 1) {
    background-color: #ffffff;
  }

  /* table caption */
  caption {
    text-align: left;
    padding: 0;
    padding-bottom: 10px;
  }

  /* table auto-number */
  table > caption > span:first-of-type {
    font-weight: bold;
    margin-right: 5px;
  }

  /* -------------------------------------------------- */
  /* code */
  /* -------------------------------------------------- */

  /* multi-line code block */
  pre {
    padding: 10px;
    background-color: #eeeeee;
    color: #000000;
    border-radius: 5px;
    break-inside: avoid;
    text-align: left;
  }

  /* inline code, ie code within normal text */
  :not(pre) > code {
    padding: 0 4px;
    background-color: #eeeeee;
    color: #000000;
    border-radius: 5px;
  }

  /* code text */
  /* apply all children, to reach syntax highlighting sub-elements */
  code,
  code * {
    /* force monospace font */
    font-family: "Source Code Pro", "Courier New", monospace;
  }

  /* -------------------------------------------------- */
  /* quotes */
  /* -------------------------------------------------- */

  /* quoted text */
  blockquote {
    margin: 0;
    padding: 0;
    border-left: 4px solid #bdbdbd;
    padding-left: 16px;
    break-inside: avoid;
  }

  /* -------------------------------------------------- */
  /* banners */
  /* -------------------------------------------------- */

  /* info banners */
  .banner {
    box-sizing: border-box;
    display: block;
    position: relative;
    width: 100%;
    margin-top: 20px;
    margin-bottom: 20px;
    padding: 20px;
    text-align: center;
  }

  /* paragraph in banner */
  .banner > p {
    margin: 0;
  }

  /* -------------------------------------------------- */
  /* highlight colors */
  /* -------------------------------------------------- */

  .white {
    background: #ffffff;
  }
  .lightgrey {
    background: #eeeeee;
  }
  .grey {
    background: #757575;
  }
  .darkgrey {
    background: #424242;
  }
  .black {
    background: #000000;
  }
  .lightred {
    background: #ffcdd2;
  }
  .lightyellow {
    background: #ffecb3;
  }
  .lightgreen {
    background: #dcedc8;
  }
  .lightblue {
    background: #e3f2fd;
  }
  .lightpurple {
    background: #f3e5f5;
  }
  .red {
    background: #f44336;
  }
  .orange {
    background: #ff9800;
  }
  .yellow {
    background: #ffeb3b;
  }
  .green {
    background: #4caf50;
  }
  .blue {
    background: #2196f3;
  }
  .purple {
    background: #9c27b0;
  }
  .white,
  .lightgrey,
  .lightred,
  .lightyellow,
  .lightgreen,
  .lightblue,
  .lightpurple,
  .orange,
  .yellow,
  .white a,
  .lightgrey a,
  .lightred a,
  .lightyellow a,
  .lightgreen a,
  .lightblue a,
  .lightpurple a,
  .orange a,
  .yellow a {
    color: #000000;
  }
  .grey,
  .darkgrey,
  .black,
  .red,
  .green,
  .blue,
  .purple,
  .grey a,
  .darkgrey a,
  .black a,
  .red a,
  .green a,
  .blue a,
  .purple a {
    color: #ffffff;
  }

  /* -------------------------------------------------- */
  /* buttons */
  /* -------------------------------------------------- */

  /* class for styling links like buttons */
  .button {
    display: inline-flex;
    justify-content: center;
    align-items: center;
    margin: 5px;
    padding: 10px 20px;
    font-size: 0.75em;
    font-weight: 600;
    text-transform: uppercase;
    text-decoration: none;
    letter-spacing: 1px;
    background: none;
    color: #2196f3;
    border: solid 1px #bdbdbd;
    border-radius: 5px;
  }

  /* buttons when hovered */
  .button:hover:not([disabled]),
  .icon_button:hover:not([disabled]) {
    cursor: pointer;
    background: #f5f5f5;
  }

  /* buttons when disabled */
  .button[disabled],
  .icon_button[disabled] {
    opacity: 0.35;
    pointer-events: none;
  }

  /* class for styling buttons containg only single icon */
  .icon_button {
    display: inline-flex;
    justify-content: center;
    align-items: center;
    text-decoration: none;
    margin: 0;
    padding: 0;
    background: none;
    border-radius: 5px;
    border: none;
    width: 20px;
    height: 20px;
    min-width: 20px;
    min-height: 20px;
  }

  /* icon button inner svg image */
  .icon_button > svg {
    height: 16px;
  }

  /* -------------------------------------------------- */
  /* icons */
  /* -------------------------------------------------- */

  /* class for styling icons inline with text */
  .inline_icon {
    height: 1em;
    position: relative;
    top: 0.125em;
  }

  /* -------------------------------------------------- */
  /* references */
  /* -------------------------------------------------- */

  .csl-entry {
    margin-top: 15px;
    margin-bottom: 15px;
  }

  /* -------------------------------------------------- */
  /* print control */
  /* -------------------------------------------------- */

  @media print {
    @page {
      /* suggested printing margin */
      margin: 0.5in;
    }

    /* document and "page" elements */
    html,
    body {
      margin: 0;
      padding: 0;
      width: 100%;
      height: 100%;
    }

    /* "page" element */
    body {
      font-size: 11pt !important;
      line-height: 1.35;
    }

    /* all headings */
    h1,
    h2,
    h3,
    h4,
    h5,
    h6 {
      margin: 15px 0;
    }

    /* figures and tables */
    figure,
    table {
      font-size: 0.85em;
    }

    /* table cells */
    th,
    td {
      padding: 5px;
    }

    /* shrink font awesome icons */
    i.fas,
    i.fab,
    i.far,
    i.fal {
      transform: scale(0.85);
    }

    /* decrease banner margins */
    .banner {
      margin-top: 15px;
      margin-bottom: 15px;
      padding: 15px;
    }

    /* class for centering an element vertically on its own page */
    .page_center {
      margin: auto;
      width: 100%;
      height: 100%;
      display: flex;
      align-items: center;
      vertical-align: middle;
      break-before: page;
      break-after: page;
    }

    /* always insert a page break before the element */
    .page_break_before {
      break-before: page;
    }

    /* always insert a page break after the element */
    .page_break_after {
      break-after: page;
    }

    /* avoid page break before the element */
    .page_break_before_avoid {
      break-before: avoid;
    }

    /* avoid page break after the element */
    .page_break_after_avoid {
      break-after: avoid;
    }

    /* avoid page break inside the element */
    .page_break_inside_avoid {
      break-inside: avoid;
    }
  }

  /* -------------------------------------------------- */
  /* override pandoc css quirks */
  /* -------------------------------------------------- */

  .sourceCode {
    /* prevent unsightly overflow in wide code blocks */
    overflow: auto !important;
  }

  div.sourceCode {
    /* prevent background fill on top-most code block  container */
    background: none !important;
  }

  .sourceCode * {
    /* force consistent line spacing */
    line-height: 1.5 !important;
  }

  div.sourceCode {
    /* style code block margins same as <pre> element */
    margin-top: 20px;
    margin-bottom: 20px;
  }

  /* -------------------------------------------------- */
  /* tablenos */
  /* -------------------------------------------------- */

  /* tablenos wrapper */
  .tablenos {
    width: 100%;
    margin: 20px 0;
  }

  .tablenos > table {
    /* move margins from table to table_wrapper to allow margin collapsing */
    margin: 0;
  }

  @media only screen {
    /* tablenos wrapper */
    .tablenos {
      /* show scrollbar on tables if necessary to prevent overflow */
      overflow-x: auto !important;
    }

    .tablenos th,
    .tablenos td {
      overflow-wrap: unset !important;
      word-break: unset !important;
    }

    /* table in wrapper */
    .tablenos table,
    .tablenos table * {
      /* don't break table words */
      overflow-wrap: normal !important;
    }
  }
</style>
<!-- 
    Plugin Core

    Functions needed for and shared across all first-party plugins.
-->

<script>
  // get element that is target of hash (from link element or url)
  function getHashTarget(link) {
    const hash = link ? link.hash : window.location.hash;
    const id = hash.slice(1);
    let target = document.querySelector(`[id="${id}"]`);
    if (!target) return;

    // if figure or table, modify target to get expected element
    if (id.indexOf("fig:") === 0) target = target.querySelector("figure");
    if (id.indexOf("tbl:") === 0) target = target.querySelector("table");

    return target;
  }

  // get position/dimensions of element or viewport
  function getRectInView(element) {
    let rect = {};
    rect.left = 0;
    rect.top = 0;
    rect.right = document.documentElement.clientWidth;
    rect.bottom = document.documentElement.clientHeight;
    let style = {};

    if (element instanceof HTMLElement) {
      rect = element.getBoundingClientRect();
      style = window.getComputedStyle(element);
    }

    const margin = {};
    margin.left = parseFloat(style.marginLeftWidth) || 0;
    margin.top = parseFloat(style.marginTopWidth) || 0;
    margin.right = parseFloat(style.marginRightWidth) || 0;
    margin.bottom = parseFloat(style.marginBottomWidth) || 0;

    const border = {};
    border.left = parseFloat(style.borderLeftWidth) || 0;
    border.top = parseFloat(style.borderTopWidth) || 0;
    border.right = parseFloat(style.borderRightWidth) || 0;
    border.bottom = parseFloat(style.borderBottomWidth) || 0;

    const newRect = {};
    newRect.left = rect.left + margin.left + border.left;
    newRect.top = rect.top + margin.top + border.top;
    newRect.right = rect.right + margin.right + border.right;
    newRect.bottom = rect.bottom + margin.bottom + border.bottom;
    newRect.width = newRect.right - newRect.left;
    newRect.height = newRect.bottom - newRect.top;

    return newRect;
  }

  // get position of element relative to page
  function getRectInPage(element) {
    const rect = getRectInView(element);
    const body = getRectInView(document.body);

    const newRect = {};
    newRect.left = rect.left - body.left;
    newRect.top = rect.top - body.top;
    newRect.right = rect.right - body.left;
    newRect.bottom = rect.bottom - body.top;
    newRect.width = rect.width;
    newRect.height = rect.height;

    return newRect;
  }

  // get closest element before specified element that matches query
  function firstBefore(element, query) {
    while (element && element !== document.body && !element.matches(query))
      element = element.previousElementSibling || element.parentNode;

    return element;
  }

  // check if element is part of collapsed heading
  function isCollapsed(element) {
    while (element && element !== document.body) {
      if (element.dataset.collapsed === "true") return true;
      element = element.parentNode;
    }
    return false;
  }

  // expand any collapsed parent containers of element if necessary
  function expandElement(element) {
    if (isCollapsed(element)) {
      // accordion plugin
      const heading = firstBefore(element, "h2");
      if (heading) heading.click();
      // details/summary HTML element
      const summary = firstBefore(element, "summary");
      if (summary) summary.click();
    }
  }

  // scroll to and focus element
  function goToElement(element, offset) {
    // expand accordion section if collapsed
    expandElement(element);
    const y =
      getRectInView(element).top -
      getRectInView(document.documentElement).top -
      (offset || 0);

    // trigger any function listening for "onscroll" event
    window.dispatchEvent(new Event("scroll"));
    window.scrollTo(0, y);
    document.activeElement.blur();
    element.focus();
  }

  // get list of elements after a start element up to element matching query
  function nextUntil(element, query, exclude) {
    const elements = [];
    while (((element = element.nextElementSibling), element)) {
      if (element.matches(query)) break;
      if (!element.matches(exclude)) elements.push(element);
    }
    return elements;
  }
</script>
<!--
  Accordion Plugin

  Allows sections of content under h2 headings to be collapsible.
-->

<script type="module">
  // whether to always start expanded ('false'), always start collapsed
  // ('true'), or start collapsed when screen small ('auto')
  const startCollapsed = "auto";

  // start script
  function start() {
    // run through each <h2> heading
    const headings = document.querySelectorAll("h2");
    for (const heading of headings) {
      addArrow(heading);

      // start expanded/collapsed based on option
      if (
        startCollapsed === "true" ||
        (startCollapsed === "auto" && isSmallScreen()) ||
        heading.dataset.collapsed === "true"
      )
        collapseHeading(heading);
      else expandElement(heading);
    }

    // attach hash change listener to window
    window.addEventListener("hashchange", onHashChange);
  }

  // when hash (eg manuscript.html#introduction) changes
  function onHashChange() {
    const target = getHashTarget();
    if (target) goToElement(target);
  }

  // add arrow to heading
  function addArrow(heading) {
    // add arrow button
    const arrow = document.createElement("button");
    arrow.innerHTML = document.querySelector(".icon_angle_down").innerHTML;
    arrow.classList.add("icon_button", "accordion_arrow");
    heading.insertBefore(arrow, heading.firstChild);

    // attach click listener to heading and button
    heading.addEventListener("click", onHeadingClick);
    arrow.addEventListener("click", onArrowClick);
  }

  // determine if on mobile-like device with small screen
  function isSmallScreen() {
    return Math.min(window.innerWidth, window.innerHeight) < 480;
  }

  // when <h2> heading is clicked
  function onHeadingClick(event) {
    // only collapse if <h2> itself is target of click (eg, user did
    // not click on anchor within <h2>)
    if (event.target === this) toggleCollapse(this);
  }

  // when arrow button is clicked
  function onArrowClick() {
    toggleCollapse(this.parentNode);
  }

  // collapse section if expanded, expand if collapsed
  function toggleCollapse(heading) {
    if (heading.dataset.collapsed === "false") collapseHeading(heading);
    else expandElement(heading);
  }

  // elements to exclude from collapse, such as table of contents panel,
  // hypothesis panel, etc
  const exclude = "#toc_panel, div.annotator-frame, #lightbox_overlay";

  // collapse section
  function collapseHeading(heading) {
    heading.setAttribute("data-collapsed", "true");
    const children = getChildren(heading);
    for (const child of children) child.setAttribute("data-collapsed", "true");
  }

  // expand section
  function expandElement(heading) {
    heading.setAttribute("data-collapsed", "false");
    const children = getChildren(heading);
    for (const child of children) child.setAttribute("data-collapsed", "false");
  }

  // get list of elements between this <h2> and next <h2> or <h1>
  // ("children" of the <h2> section)
  function getChildren(heading) {
    return nextUntil(heading, "h2, h1", exclude);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- angle down icon -->

<template class="icon_angle_down">
  <!-- modified from: https://fontawesome.com/icons/angle-down -->
  <svg width="16" height="16" viewBox="0 0 448 512">
    <path
      fill="currentColor"
      d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* accordion arrow button */
    .accordion_arrow {
      margin-right: 10px;
    }

    /* arrow icon when <h2> data-collapsed attribute true */
    h2[data-collapsed="true"] > .accordion_arrow > svg {
      transform: rotate(-90deg);
    }

    /* all elements (except <h2>'s) when data-collapsed attribute true */
    *:not(h2)[data-collapsed="true"] {
      display: none;
    }

    /* accordion arrow button when hovered and <h2>'s when hovered */
    .accordion_arrow:hover,
    h2[data-collapsed="true"]:hover,
    h2[data-collapsed="false"]:hover {
      cursor: pointer;
    }
  }

  /* always hide accordion arrow button on print */
  @media only print {
    .accordion_arrow {
      display: none;
    }
  }
</style>
<!--
  Anchors Plugin

  Adds an anchor next to each of a certain type of element that provides a
  human-readable url to that specific item/position in the document (e.g.
  "manuscript.html#abstract"). It also makes it such that scrolling out of view
  of a target removes its identifier from the url.
-->

<script type="module">
  // which types of elements to add anchors next to, in "document.querySelector"
  // format
  const typesQuery =
    'h1, h2, h3, div[id^="fig:"], div[id^="tbl:"], span[id^="eq:"]';

  // start script
  function start() {
    // add anchor to each element of specified types
    const elements = document.querySelectorAll(typesQuery);
    for (const element of elements) addAnchor(element);

    // attach scroll listener to window
    window.addEventListener("scroll", onScroll);
  }

  // when window is scrolled
  function onScroll() {
    // if url has hash and user has scrolled out of view of hash
    // target, remove hash from url
    const tolerance = 100;
    const target = getHashTarget();
    if (target) {
      if (
        target.getBoundingClientRect().top > window.innerHeight + tolerance ||
        target.getBoundingClientRect().bottom < 0 - tolerance
      )
        history.pushState(null, null, " ");
    }
  }

  // add anchor to element
  function addAnchor(element) {
    let addTo; // element to add anchor button to

    // if figure or table, modify withId and addTo to get expected
    // elements
    if (element.id.indexOf("fig:") === 0) {
      addTo = element.querySelector("figcaption");
    } else if (element.id.indexOf("tbl:") === 0) {
      addTo = element.querySelector("caption");
    } else if (element.id.indexOf("eq:") === 0) {
      addTo = element.querySelector(".eqnos-number");
    }

    addTo = addTo || element;
    const id = element.id || null;

    // do not add anchor if element doesn't have assigned id.
    // id is generated by pandoc and is assumed to be unique and
    // human-readable
    if (!id) return;

    // create anchor button
    const anchor = document.createElement("a");
    anchor.innerHTML = document.querySelector(".icon_link").innerHTML;
    anchor.title = "Link to this part of the document";
    anchor.classList.add("icon_button", "anchor");
    anchor.dataset.ignore = "true";
    anchor.href = "#" + id;
    addTo.appendChild(anchor);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- link icon -->

<template class="icon_link">
  <!-- modified from: https://fontawesome.com/icons/link -->
  <svg width="16" height="16" viewBox="0 0 512 512">
    <path
      fill="currentColor"
      d="M326.612 185.391c59.747 59.809 58.927 155.698.36 214.59-.11.12-.24.25-.36.37l-67.2 67.2c-59.27 59.27-155.699 59.262-214.96 0-59.27-59.26-59.27-155.7 0-214.96l37.106-37.106c9.84-9.84 26.786-3.3 27.294 10.606.648 17.722 3.826 35.527 9.69 52.721 1.986 5.822.567 12.262-3.783 16.612l-13.087 13.087c-28.026 28.026-28.905 73.66-1.155 101.96 28.024 28.579 74.086 28.749 102.325.51l67.2-67.19c28.191-28.191 28.073-73.757 0-101.83-3.701-3.694-7.429-6.564-10.341-8.569a16.037 16.037 0 0 1-6.947-12.606c-.396-10.567 3.348-21.456 11.698-29.806l21.054-21.055c5.521-5.521 14.182-6.199 20.584-1.731a152.482 152.482 0 0 1 20.522 17.197zM467.547 44.449c-59.261-59.262-155.69-59.27-214.96 0l-67.2 67.2c-.12.12-.25.25-.36.37-58.566 58.892-59.387 154.781.36 214.59a152.454 152.454 0 0 0 20.521 17.196c6.402 4.468 15.064 3.789 20.584-1.731l21.054-21.055c8.35-8.35 12.094-19.239 11.698-29.806a16.037 16.037 0 0 0-6.947-12.606c-2.912-2.005-6.64-4.875-10.341-8.569-28.073-28.073-28.191-73.639 0-101.83l67.2-67.19c28.239-28.239 74.3-28.069 102.325.51 27.75 28.3 26.872 73.934-1.155 101.96l-13.087 13.087c-4.35 4.35-5.769 10.79-3.783 16.612 5.864 17.194 9.042 34.999 9.69 52.721.509 13.906 17.454 20.446 27.294 10.606l37.106-37.106c59.271-59.259 59.271-155.699.001-214.959z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* anchor button */
    .anchor {
      opacity: 0;
      margin-left: 5px;
    }

    /* anchor buttons within <h2>'s */
    h2 .anchor {
      margin-left: 10px;
    }

    /* anchor buttons when hovered/focused and anything containing an anchor button when hovered */
    *:hover > .anchor,
    .anchor:hover,
    .anchor:focus {
      opacity: 1;
    }

    /* anchor button when hovered */
    .anchor:hover {
      cursor: pointer;
    }
  }

  /* always show anchor button on devices with no mouse/hover ability */
  @media (hover: none) {
    .anchor {
      opacity: 1;
    }
  }

  /* always hide anchor button on print */
  @media only print {
    .anchor {
      display: none;
    }
  }
</style>
<!-- 
    Attributes Plugin

    Allows arbitrary HTML attributes to be attached to (almost) any element.
    Place an HTML comment inside or next to the desired element with the content:
    $attribute="value"
-->

<script type="module">
  // start script
  function start() {
    // get list of comments in document
    const comments = findComments();

    for (const comment of comments)
      if (comment.parentElement)
        addAttributes(comment.parentElement, comment.nodeValue.trim());
  }

  // add html attributes to specified element based on string of
  // html attributes and values
  function addAttributes(element, text) {
    // regex's for finding attribute/value pairs in the format of
    // attribute="value" or attribute='value
    const regex2 = /\$([a-zA-Z\-]+)?=\"(.+?)\"/;
    const regex1 = /\$([a-zA-Z\-]+)?=\'(.+?)\'/;

    // loop through attribute/value pairs
    let match;
    while ((match = text.match(regex2) || text.match(regex1))) {
      // get attribute and value from regex capture groups
      let attribute = match[1];
      let value = match[2];

      // remove from string
      text = text.substring(match.index + match[0].length);

      if (!attribute || !value) break;

      // set attribute of parent element
      try {
        element.setAttribute(attribute, value);
      } catch (error) {
        console.log(error);
      }

      // special case for colspan
      if (attribute === "colspan") removeTableCells(element, value);
    }
  }

  // get list of comment elements in document
  function findComments() {
    const comments = [];

    // iterate over comment nodes in document
    function acceptNode(node) {
      return NodeFilter.FILTER_ACCEPT;
    }
    const iterator = document.createNodeIterator(
      document.body,
      NodeFilter.SHOW_COMMENT,
      acceptNode
    );
    let node;
    while ((node = iterator.nextNode())) comments.push(node);

    return comments;
  }

  // remove certain number of cells after specified cell
  function removeTableCells(cell, number) {
    number = parseInt(number);
    if (!number) return;

    // remove elements
    for (; number > 1; number--) {
      if (cell.nextElementSibling) cell.nextElementSibling.remove();
    }
  }

  // start script on DOMContentLoaded instead of load to ensure this plugins
  // runs before other plugins
  window.addEventListener("DOMContentLoaded", start);
</script>
<!--
  Jump to First Plugin

  Adds a button next to each reference entry, figure, and table that jumps the
  page to the first occurrence of a link to that item in the manuscript.
-->

<script type="module">
  // whether to add buttons next to reference entries
  const references = "true";
  // whether to add buttons next to figures
  const figures = "true";
  // whether to add buttons next to tables
  const tables = "true";

  // start script
  function start() {
    if (references !== "false")
      makeButtons(`div[id^="ref-"]`, ".csl-left-margin", "reference");
    if (figures !== "false")
      makeButtons(`div[id^="fig:"]`, "figcaption", "figure");
    if (tables !== "false") makeButtons(`div[id^="tbl:"]`, "caption", "table");
  }

  // when jump button clicked
  function onButtonClick() {
    const first = getFirstOccurrence(this.dataset.id);
    if (!first) return;

    // update url hash so navigating "back" in history will return user to button
    window.location.hash = this.dataset.id;
    // scroll to link
    const timeout = function () {
      goToElement(first, window.innerHeight * 0.5);
    };
    window.setTimeout(timeout, 0);
  }

  // get first occurrence of link to item in document
  function getFirstOccurrence(id) {
    let query = "a";
    query += '[href="#' + id + '"]';
    // exclude buttons, anchor links, toc links, etc
    query += ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    return document.querySelector(query);
  }

  // add button next to each reference entry, figure, or table
  function makeButtons(query, containerQuery, subject) {
    const elements = document.querySelectorAll(query);
    for (const element of elements) {
      const id = element.id;
      const buttonContainer = element.querySelector(containerQuery);
      const first = getFirstOccurrence(id);

      // if can't find link to reference or place to put button, ignore
      if (!first || !buttonContainer) continue;

      // make jump button
      let button = document.createElement("button");
      button.classList.add("icon_button", "jump_arrow");
      button.title = `Jump to the first occurrence of this ${subject} in the document`;
      const icon = document.querySelector(".icon_angle_double_up");
      button.innerHTML = icon.innerHTML;
      button.dataset.id = id;
      button.dataset.ignore = "true";
      button.addEventListener("click", onButtonClick);
      buttonContainer.prepend(button);
    }
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- angle double up icon -->

<template class="icon_angle_double_up">
  <!-- modified from: https://fontawesome.com/icons/angle-double-up -->
  <svg width="16" height="16" viewBox="0 0 320 512">
    <path
      fill="currentColor"
      d="M177 255.7l136 136c9.4 9.4 9.4 24.6 0 33.9l-22.6 22.6c-9.4 9.4-24.6 9.4-33.9 0L160 351.9l-96.4 96.4c-9.4 9.4-24.6 9.4-33.9 0L7 425.7c-9.4-9.4-9.4-24.6 0-33.9l136-136c9.4-9.5 24.6-9.5 34-.1zm-34-192L7 199.7c-9.4 9.4-9.4 24.6 0 33.9l22.6 22.6c9.4 9.4 24.6 9.4 33.9 0l96.4-96.4 96.4 96.4c9.4 9.4 24.6 9.4 33.9 0l22.6-22.6c9.4-9.4 9.4-24.6 0-33.9l-136-136c-9.2-9.4-24.4-9.4-33.8 0z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* jump button */
    .jump_arrow {
      position: relative;
      top: 0.125em;
      margin-right: 5px;
    }
  }

  /* always hide jump button on print */
  @media only print {
    .jump_arrow {
      display: none;
    }
  }
</style>
<!-- 
    Lightbox Plugin

    Makes it such that when a user clicks on an image, the image fills the
    screen and the user can pan/drag/zoom the image and navigate between other
    images in the document.
-->

<script type="module">
  // list of possible zoom/scale factors
  const zooms =
    "0.1, 0.25, 0.333333, 0.5, 0.666666, 0.75, 1, 1.25, 1.5, 1.75, 2, 2.5, 3, 3.5, 4, 5, 6, 7, 8";
  // whether to fit image to view ('fit'), display at 100% and shrink if
  // necessary ('shrink'), or always display at 100% ('100')
  const defaultZoom = "fit";
  // whether to zoom in/out toward center of view ('true') or mouse ('false')
  const centerZoom = "false";

  // start script
  function start() {
    // run through each <img> element
    const imgs = document.querySelectorAll("figure > img");
    let count = 1;
    for (const img of imgs) {
      img.classList.add("lightbox_document_img");
      img.dataset.number = count;
      img.dataset.total = imgs.length;
      img.addEventListener("click", openLightbox);
      count++;
    }

    // attach mouse and key listeners to window
    window.addEventListener("mousemove", onWindowMouseMove);
    window.addEventListener("keyup", onKeyUp);
  }

  // when mouse is moved anywhere in window
  function onWindowMouseMove(event) {
    window.mouseX = event.clientX;
    window.mouseY = event.clientY;
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    switch (event.key) {
      // trigger click of prev button
      case "ArrowLeft":
        const prevButton = document.getElementById("lightbox_prev_button");
        if (prevButton) prevButton.click();
        break;
      // trigger click of next button
      case "ArrowRight":
        const nextButton = document.getElementById("lightbox_next_button");
        if (nextButton) nextButton.click();
        break;
      // close on esc
      case "Escape":
        closeLightbox();
        break;
    }
  }

  // open lightbox
  function openLightbox() {
    const lightbox = makeLightbox(this);
    if (!lightbox) return;

    blurBody(lightbox);
    document.body.appendChild(lightbox);
  }

  // make lightbox
  function makeLightbox(img) {
    // delete lightbox if it exists, start fresh
    closeLightbox();

    // create screen overlay containing lightbox
    const overlay = document.createElement("div");
    overlay.id = "lightbox_overlay";

    // create image info boxes
    const numberInfo = document.createElement("div");
    const zoomInfo = document.createElement("div");
    numberInfo.id = "lightbox_number_info";
    zoomInfo.id = "lightbox_zoom_info";

    // create container for image
    const imageContainer = document.createElement("div");
    imageContainer.id = "lightbox_image_container";
    const lightboxImg = makeLightboxImg(
      img,
      imageContainer,
      numberInfo,
      zoomInfo
    );
    imageContainer.appendChild(lightboxImg);

    // create bottom container for caption and navigation buttons
    const bottomContainer = document.createElement("div");
    bottomContainer.id = "lightbox_bottom_container";
    const caption = makeCaption(img);
    const prevButton = makePrevButton(img);
    const nextButton = makeNextButton(img);
    bottomContainer.appendChild(prevButton);
    bottomContainer.appendChild(caption);
    bottomContainer.appendChild(nextButton);

    // attach top middle and bottom to overlay
    overlay.appendChild(numberInfo);
    overlay.appendChild(zoomInfo);
    overlay.appendChild(imageContainer);
    overlay.appendChild(bottomContainer);

    return overlay;
  }

  // make <img> object that is intuitively draggable and zoomable
  function makeLightboxImg(sourceImg, container, numberInfoBox, zoomInfoBox) {
    // create copy of source <img>
    const img = sourceImg.cloneNode(true);
    img.classList.remove("lightbox_document_img");
    img.removeAttribute("id");
    img.removeAttribute("width");
    img.removeAttribute("height");
    img.style.position = "unset";
    img.style.margin = "0";
    img.style.padding = "0";
    img.style.width = "";
    img.style.height = "";
    img.style.minWidth = "";
    img.style.minHeight = "";
    img.style.maxWidth = "";
    img.style.maxHeight = "";
    img.id = "lightbox_img";

    // build sorted list of zoomSteps
    const zoomSteps = zooms.split(/[^0-9.]/).map((step) => parseFloat(step));
    zoomSteps.sort((a, b) => a - b);

    // <img> object property variables
    let zoom = 1;
    let translateX = 0;
    let translateY = 0;
    let clickMouseX = undefined;
    let clickMouseY = undefined;
    let clickTranslateX = undefined;
    let clickTranslateY = undefined;

    updateNumberInfo();

    // update image numbers displayed in info box
    function updateNumberInfo() {
      numberInfoBox.innerHTML =
        sourceImg.dataset.number + " of " + sourceImg.dataset.total;
    }

    // update zoom displayed in info box
    function updateZoomInfo() {
      let zoomInfo = zoom * 100;
      if (!Number.isInteger(zoomInfo)) zoomInfo = zoomInfo.toFixed(2);
      zoomInfoBox.innerHTML = zoomInfo + "%";
    }

    // move to closest zoom step above current zoom
    const zoomIn = function () {
      for (const zoomStep of zoomSteps) {
        if (zoomStep > zoom) {
          zoom = zoomStep;
          break;
        }
      }
      updateTransform();
    };

    // move to closest zoom step above current zoom
    const zoomOut = function () {
      zoomSteps.reverse();
      for (const zoomStep of zoomSteps) {
        if (zoomStep < zoom) {
          zoom = zoomStep;
          break;
        }
      }
      zoomSteps.reverse();

      updateTransform();
    };

    // update display of <img> based on scale/translate properties
    const updateTransform = function () {
      // set transform
      img.style.transform =
        "translate(" +
        (translateX || 0) +
        "px," +
        (translateY || 0) +
        "px) scale(" +
        (zoom || 1) +
        ")";

      // get new width/height after scale
      const rect = img.getBoundingClientRect();
      // limit translate
      translateX = Math.max(translateX, -rect.width / 2);
      translateX = Math.min(translateX, rect.width / 2);
      translateY = Math.max(translateY, -rect.height / 2);
      translateY = Math.min(translateY, rect.height / 2);

      // set transform
      img.style.transform =
        "translate(" +
        (translateX || 0) +
        "px," +
        (translateY || 0) +
        "px) scale(" +
        (zoom || 1) +
        ")";

      updateZoomInfo();
    };

    // fit <img> to container
    const fit = function () {
      // no x/y offset, 100% zoom by default
      translateX = 0;
      translateY = 0;
      zoom = 1;

      // widths of <img> and container
      const imgWidth = img.naturalWidth;
      const imgHeight = img.naturalHeight;
      const containerWidth = parseFloat(
        window.getComputedStyle(container).width
      );
      const containerHeight = parseFloat(
        window.getComputedStyle(container).height
      );

      // how much zooming is needed to fit <img> to container
      const xRatio = imgWidth / containerWidth;
      const yRatio = imgHeight / containerHeight;
      const maxRatio = Math.max(xRatio, yRatio);
      const newZoom = 1 / maxRatio;

      // fit <img> to container according to option
      if (defaultZoom === "shrink") {
        if (maxRatio > 1) zoom = newZoom;
      } else if (defaultZoom === "fit") zoom = newZoom;

      updateTransform();
    };

    // when mouse wheel is rolled anywhere in container
    const onContainerWheel = function (event) {
      if (!event) return;

      // let ctrl + mouse wheel to zoom behave as normal
      if (event.ctrlKey) return;

      // prevent normal scroll behavior
      event.preventDefault();
      event.stopPropagation();

      // point around which to scale img
      const viewRect = container.getBoundingClientRect();
      const viewX = (viewRect.left + viewRect.right) / 2;
      const viewY = (viewRect.top + viewRect.bottom) / 2;
      const originX = centerZoom === "true" ? viewX : mouseX;
      const originY = centerZoom === "true" ? viewY : mouseY;

      // get point on image under origin
      const oldRect = img.getBoundingClientRect();
      const oldPercentX = (originX - oldRect.left) / oldRect.width;
      const oldPercentY = (originY - oldRect.top) / oldRect.height;

      // increment/decrement zoom
      if (event.deltaY < 0) zoomIn();
      if (event.deltaY > 0) zoomOut();

      // get offset between previous image point and origin
      const newRect = img.getBoundingClientRect();
      const offsetX = originX - (newRect.left + newRect.width * oldPercentX);
      const offsetY = originY - (newRect.top + newRect.height * oldPercentY);

      // translate image to keep image point under origin
      translateX += offsetX;
      translateY += offsetY;

      // perform translate
      updateTransform();
    };

    // when container is clicked
    function onContainerClick(event) {
      // if container itself is target of click, and not other
      // element above it
      if (event.target === this) closeLightbox();
    }

    // when mouse button is pressed on image
    const onImageMouseDown = function (event) {
      // store original mouse position relative to image
      clickMouseX = window.mouseX;
      clickMouseY = window.mouseY;
      clickTranslateX = translateX;
      clickTranslateY = translateY;
      event.stopPropagation();
      event.preventDefault();
    };

    // when mouse button is released anywhere in window
    const onWindowMouseUp = function (event) {
      // reset original mouse position
      clickMouseX = undefined;
      clickMouseY = undefined;
      clickTranslateX = undefined;
      clickTranslateY = undefined;

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("mouseup", onWindowMouseUp);
    };

    // when mouse is moved anywhere in window
    const onWindowMouseMove = function (event) {
      if (
        clickMouseX === undefined ||
        clickMouseY === undefined ||
        clickTranslateX === undefined ||
        clickTranslateY === undefined
      )
        return;

      // offset image based on original and current mouse position
      translateX = clickTranslateX + window.mouseX - clickMouseX;
      translateY = clickTranslateY + window.mouseY - clickMouseY;
      updateTransform();
      event.preventDefault();

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("mousemove", onWindowMouseMove);
    };

    // when window is resized
    const onWindowResize = function (event) {
      fit();

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("resize", onWindowResize);
    };

    // attach the necessary event listeners
    img.addEventListener("dblclick", fit);
    img.addEventListener("mousedown", onImageMouseDown);
    container.addEventListener("wheel", onContainerWheel);
    container.addEventListener("mousedown", onContainerClick);
    container.addEventListener("touchstart", onContainerClick);
    window.addEventListener("mouseup", onWindowMouseUp);
    window.addEventListener("mousemove", onWindowMouseMove);
    window.addEventListener("resize", onWindowResize);

    // run fit() after lightbox atttached to document and <img> Loaded
    // so needed container and img dimensions available
    img.addEventListener("load", fit);

    return img;
  }

  // make caption
  function makeCaption(img) {
    const caption = document.createElement("div");
    caption.id = "lightbox_caption";
    const captionSource = img.nextElementSibling;
    if (captionSource.tagName.toLowerCase() === "figcaption") {
      const captionCopy = makeCopy(captionSource);
      caption.innerHTML = captionCopy.innerHTML;
    }

    caption.addEventListener("touchstart", function (event) {
      event.stopPropagation();
    });

    return caption;
  }

  // make carbon copy of html dom element
  function makeCopy(source) {
    const sourceCopy = source.cloneNode(true);

    // delete elements marked with ignore (eg anchor and jump buttons)
    const deleteFromCopy = sourceCopy.querySelectorAll('[data-ignore="true"]');
    for (const element of deleteFromCopy) element.remove();

    // delete certain element attributes
    const attributes = [
      "id",
      "data-collapsed",
      "data-selected",
      "data-highlighted",
      "data-glow",
    ];
    for (const attribute of attributes) {
      sourceCopy.removeAttribute(attribute);
      const elements = sourceCopy.querySelectorAll("[" + attribute + "]");
      for (const element of elements) element.removeAttribute(attribute);
    }

    return sourceCopy;
  }

  // make button to jump to previous image in document
  function makePrevButton(img) {
    const prevButton = document.createElement("button");
    prevButton.id = "lightbox_prev_button";
    prevButton.title = "Jump to the previous image in the document [←]";
    prevButton.classList.add("icon_button", "lightbox_button");
    prevButton.innerHTML = document.querySelector(".icon_caret_left").innerHTML;

    // attach click listeners to button
    prevButton.addEventListener("click", function () {
      getPrevImg(img).click();
    });

    return prevButton;
  }

  // make button to jump to next image in document
  function makeNextButton(img) {
    const nextButton = document.createElement("button");
    nextButton.id = "lightbox_next_button";
    nextButton.title = "Jump to the next image in the document [→]";
    nextButton.classList.add("icon_button", "lightbox_button");
    nextButton.innerHTML = document.querySelector(
      ".icon_caret_right"
    ).innerHTML;

    // attach click listeners to button
    nextButton.addEventListener("click", function () {
      getNextImg(img).click();
    });

    return nextButton;
  }

  // get previous image in document
  function getPrevImg(img) {
    const imgs = document.querySelectorAll(".lightbox_document_img");

    // find index of provided img
    let index;
    for (index = 0; index < imgs.length; index++) {
      if (imgs[index] === img) break;
    }

    // wrap index to other side if < 1
    if (index - 1 >= 0) index--;
    else index = imgs.length - 1;
    return imgs[index];
  }

  // get next image in document
  function getNextImg(img) {
    const imgs = document.querySelectorAll(".lightbox_document_img");

    // find index of provided img
    let index;
    for (index = 0; index < imgs.length; index++) {
      if (imgs[index] === img) break;
    }

    // wrap index to other side if > total
    if (index + 1 <= imgs.length - 1) index++;
    else index = 0;
    return imgs[index];
  }

  // close lightbox
  function closeLightbox() {
    focusBody();

    const lightbox = document.getElementById("lightbox_overlay");
    if (lightbox) lightbox.remove();
  }

  // make all elements behind lightbox non-focusable
  function blurBody(overlay) {
    const all = document.querySelectorAll("*");
    for (const element of all) element.tabIndex = -1;
    document.body.classList.add("body_no_scroll");
  }

  // make all elements focusable again
  function focusBody() {
    const all = document.querySelectorAll("*");
    for (const element of all) element.removeAttribute("tabIndex");
    document.body.classList.remove("body_no_scroll");
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
  <!-- modified from: https://fontawesome.com/icons/caret-left -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
    ></path>
  </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
  <!-- modified from: https://fontawesome.com/icons/caret-right -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* regular <img> in document when hovered */
    img.lightbox_document_img:hover {
      cursor: pointer;
    }

    .body_no_scroll {
      overflow: hidden !important;
    }

    /* screen overlay */
    #lightbox_overlay {
      display: flex;
      flex-direction: column;
      position: fixed;
      left: 0;
      top: 0;
      right: 0;
      bottom: 0;
      background: rgba(0, 0, 0, 0.75);
      z-index: 3;
    }

    /* middle area containing lightbox image */
    #lightbox_image_container {
      flex-grow: 1;
      display: flex;
      justify-content: center;
      align-items: center;
      overflow: hidden;
      position: relative;
      padding: 20px;
    }

    /* bottom area containing caption */
    #lightbox_bottom_container {
      display: flex;
      justify-content: center;
      align-items: center;
      height: 100px;
      min-height: 100px;
      max-height: 100px;
      background: rgba(0, 0, 0, 0.5);
    }

    /* image number info text box */
    #lightbox_number_info {
      position: absolute;
      color: #ffffff;
      font-weight: 600;
      left: 2px;
      top: 0;
      z-index: 4;
    }

    /* zoom info text box */
    #lightbox_zoom_info {
      position: absolute;
      color: #ffffff;
      font-weight: 600;
      right: 2px;
      top: 0;
      z-index: 4;
    }

    /* copy of image caption */
    #lightbox_caption {
      box-sizing: border-box;
      display: inline-block;
      width: 100%;
      max-height: 100%;
      padding: 10px 0;
      text-align: center;
      overflow-y: auto;
      color: #ffffff;
    }

    /* navigation previous/next button */
    .lightbox_button {
      width: 100px;
      height: 100%;
      min-width: 100px;
      min-height: 100%;
      color: #ffffff;
    }

    /* navigation previous/next button when hovered */
    .lightbox_button:hover {
      background: none !important;
    }

    /* navigation button icon */
    .lightbox_button > svg {
      height: 25px;
    }

    /* figure auto-number */
    #lightbox_caption > span:first-of-type {
      font-weight: bold;
      margin-right: 5px;
    }

    /* lightbox image when hovered */
    #lightbox_img:hover {
      cursor: grab;
    }

    /* lightbox image when grabbed */
    #lightbox_img:active {
      cursor: grabbing;
    }
  }

  /* when on screen < 480px wide */
  @media only screen and (max-width: 480px) {
    /* make navigation buttons skinnier on small screens to make more room for caption text */
    .lightbox_button {
      width: 50px;
      min-width: 50px;
    }
  }

  /* always hide lightbox on print */
  @media only print {
    #lightbox_overlay {
      display: none;
    }
  }
</style>
<!-- 
  Link Highlight Plugin

  Makes it such that when a user hovers or focuses a link, other links that have
  the same target will be highlighted. It also makes it such that when clicking
  a link, the target of the link (eg reference, figure, table) is briefly
  highlighted.
-->

<script type="module">
  // whether to also highlight links that go to external urls
  const externalLinks = "false";
  // whether user must click off to unhighlight instead of just
  // un-hovering
  const clickUnhighlight = "false";
  // whether to also highlight links that are unique
  const highlightUnique = "true";

  // start script
  function start() {
    const links = getLinks();
    for (const link of links) {
      // attach mouse and focus listeners to link
      link.addEventListener("mouseenter", onLinkFocus);
      link.addEventListener("focus", onLinkFocus);
      link.addEventListener("mouseleave", onLinkUnhover);
    }

    // attach click and hash change listeners to window
    window.addEventListener("click", onClick);
    window.addEventListener("touchstart", onClick);
    window.addEventListener("hashchange", onHashChange);

    // run hash change on window load in case user has navigated
    // directly to hash
    onHashChange();
  }

  // when link is focused (tabbed to) or hovered
  function onLinkFocus() {
    highlight(this);
  }

  // when link is unhovered
  function onLinkUnhover() {
    if (clickUnhighlight !== "true") unhighlightAll();
  }

  // when the mouse is clicked anywhere in window
  function onClick(event) {
    unhighlightAll();
  }

  // when hash (eg manuscript.html#introduction) changes
  function onHashChange() {
    const target = getHashTarget();
    if (target) glowElement(target);
  }

  // start glow sequence on an element
  function glowElement(element) {
    const startGlow = function () {
      onGlowEnd();
      element.dataset.glow = "true";
      element.addEventListener("animationend", onGlowEnd);
    };
    const onGlowEnd = function () {
      element.removeAttribute("data-glow");
      element.removeEventListener("animationend", onGlowEnd);
    };
    startGlow();
  }

  // highlight link and all others with same target
  function highlight(link) {
    // force unhighlight all to start fresh
    unhighlightAll();

    // get links with same target
    if (!link) return;
    const sameLinks = getSameLinks(link);

    // if link unique and option is off, exit and don't highlight
    if (sameLinks.length <= 1 && highlightUnique !== "true") return;

    // highlight all same links, and "select" (special highlight) this
    // one
    for (const sameLink of sameLinks) {
      if (sameLink === link) sameLink.setAttribute("data-selected", "true");
      else sameLink.setAttribute("data-highlighted", "true");
    }
  }

  // unhighlight all links
  function unhighlightAll() {
    const links = getLinks();
    for (const link of links) {
      link.setAttribute("data-selected", "false");
      link.setAttribute("data-highlighted", "false");
    }
  }

  // get links with same target
  function getSameLinks(link) {
    const results = [];
    const links = getLinks();
    for (const otherLink of links) {
      if (otherLink.getAttribute("href") === link.getAttribute("href"))
        results.push(otherLink);
    }
    return results;
  }

  // get all links of types we wish to handle
  function getLinks() {
    let query = "a";
    if (externalLinks !== "true") query += '[href^="#"]';
    // exclude buttons, anchor links, toc links, etc
    query += ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    return document.querySelectorAll(query);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<style>
  @media only screen {
    /* anything with data-highlighted attribute true */
    [data-highlighted="true"] {
      background: #ffeb3b;
    }

    /* anything with data-selected attribute true */
    [data-selected="true"] {
      background: #ff8a65 !important;
    }

    /* animation definition for glow */
    @keyframes highlight_glow {
      0% {
        background: none;
      }
      10% {
        background: #bbdefb;
      }
      100% {
        background: none;
      }
    }

    /* anything with data-glow attribute true */
    [data-glow="true"] {
      animation: highlight_glow 2s;
    }
  }
</style>
<!--
  Table of Contents Plugin

  Provides a "table of contents" (toc) panel on the side of the document that
  allows the user to conveniently navigate between sections of the document.
-->

<script type="module">
  // which types of elements to add links for, in "document.querySelector" format
  const typesQuery = "h1, h2, h3";
  // whether toc starts open. use 'true' or 'false', or 'auto' to
  // use 'true' behavior when screen wide enough and 'false' when not
  const startOpen = "false";
  // whether toc closes when clicking on toc link. use 'true' or
  // 'false', or 'auto' to use 'false' behavior when screen wide
  // enough and 'true' when not
  const clickClose = "auto";
  // if list item is more than this many characters, text will be
  // truncated
  const charLimit = "50";
  // whether or not to show bullets next to each toc item
  const bullets = "false";

  // start script
  function start() {
    // make toc panel and populate with entries (links to document
    // sections)
    const panel = makePanel();
    if (!panel) return;
    makeEntries(panel);
    // attach panel to document after making entries, so 'toc' heading
    // in panel isn't included in toc
    document.body.insertBefore(panel, document.body.firstChild);

    // initial panel state
    if (startOpen === "true" || (startOpen === "auto" && !isSmallScreen()))
      openPanel();
    else closePanel();

    // attach click, scroll, and hash change listeners to window
    window.addEventListener("click", onClick);
    window.addEventListener("scroll", onScroll);
    window.addEventListener("hashchange", onScroll);
    window.addEventListener("keyup", onKeyUp);
    onScroll();

    // add class to push document body down out of way of toc button
    document.body.classList.add("toc_body_nudge");
  }

  // determine if screen wide enough to fit toc panel
  function isSmallScreen() {
    // in default theme:
    // 816px = 8.5in = width of "page" (<body>) element
    // 260px = min width of toc panel (*2 for both sides of <body>)
    return window.innerWidth < 816 + 260 * 2;
  }

  // when mouse is clicked anywhere in window
  function onClick() {
    if (isSmallScreen()) closePanel();
  }

  // when window is scrolled or hash changed
  function onScroll() {
    highlightViewed();
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    // close on esc
    if (event.key === "Escape") closePanel();
  }

  // find entry of currently viewed document section in toc and highlight
  function highlightViewed() {
    const firstId = getFirstInView(typesQuery);

    // get toc entries (links), unhighlight all, then highlight viewed
    const list = document.getElementById("toc_list");
    if (!firstId || !list) return;
    const links = list.querySelectorAll("a");
    for (const link of links) link.dataset.viewing = "false";
    const link = list.querySelector('a[href="#' + firstId + '"]');
    if (!link) return;
    link.dataset.viewing = "true";
  }

  // get first or previous toc listed element in top half of view
  function getFirstInView(query) {
    // get all elements matching query and with id
    const elements = document.querySelectorAll(query);
    const elementsWithIds = [];
    for (const element of elements) {
      if (element.id) elementsWithIds.push(element);
    }

    // get first or previous element in top half of view
    for (let i = 0; i < elementsWithIds.length; i++) {
      const element = elementsWithIds[i];
      const prevElement = elementsWithIds[Math.max(0, i - 1)];
      if (element.getBoundingClientRect().top >= 0) {
        if (element.getBoundingClientRect().top < window.innerHeight / 2)
          return element.id;
        else return prevElement.id;
      }
    }
  }

  // make panel
  function makePanel() {
    // create panel
    const panel = document.createElement("div");
    panel.id = "toc_panel";
    if (bullets === "true") panel.dataset.bullets = "true";

    // create header
    const header = document.createElement("div");
    header.id = "toc_header";

    // create toc button
    const button = document.createElement("button");
    button.id = "toc_button";
    button.innerHTML = document.querySelector(".icon_th_list").innerHTML;
    button.title = "Table of Contents";
    button.classList.add("icon_button");

    // create header text
    const text = document.createElement("h4");
    text.innerHTML = "Table of Contents";

    // create container for toc list
    const list = document.createElement("div");
    list.id = "toc_list";

    // attach click listeners
    panel.addEventListener("click", onPanelClick);
    header.addEventListener("click", onHeaderClick);
    button.addEventListener("click", onButtonClick);

    // attach elements
    header.appendChild(button);
    header.appendChild(text);
    panel.appendChild(header);
    panel.appendChild(list);

    return panel;
  }

  // create toc entries (links) to each element of the specified types
  function makeEntries(panel) {
    const elements = document.querySelectorAll(typesQuery);
    for (const element of elements) {
      // do not add link if element doesn't have assigned id
      if (!element.id) continue;

      // create link/list item
      const link = document.createElement("a");
      link.classList.add("toc_link");
      switch (element.tagName.toLowerCase()) {
        case "h1":
          link.dataset.level = "1";
          break;
        case "h2":
          link.dataset.level = "2";
          break;
        case "h3":
          link.dataset.level = "3";
          break;
        case "h4":
          link.dataset.level = "4";
          break;
      }
      link.title = element.innerText;
      let text = element.innerText;
      if (text.length > charLimit) text = text.slice(0, charLimit) + "...";
      link.innerHTML = text;
      link.href = "#" + element.id;
      link.addEventListener("click", onLinkClick);

      // attach link
      panel.querySelector("#toc_list").appendChild(link);
    }
  }

  // when panel is clicked
  function onPanelClick(event) {
    // stop click from propagating to window/document and closing panel
    event.stopPropagation();
  }

  // when header itself is clicked
  function onHeaderClick(event) {
    togglePanel();
  }

  // when button is clicked
  function onButtonClick(event) {
    togglePanel();
    // stop header underneath button from also being clicked
    event.stopPropagation();
  }

  // when link is clicked
  function onLinkClick(event) {
    if (clickClose === "true" || (clickClose === "auto" && isSmallScreen()))
      closePanel();
    else openPanel();
  }

  // open panel if closed, close if opened
  function togglePanel() {
    const panel = document.getElementById("toc_panel");
    if (!panel) return;

    if (panel.dataset.open === "true") closePanel();
    else openPanel();
  }

  // open panel
  function openPanel() {
    const panel = document.getElementById("toc_panel");
    if (panel) panel.dataset.open = "true";
  }

  // close panel
  function closePanel() {
    const panel = document.getElementById("toc_panel");
    if (panel) panel.dataset.open = "false";
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- th list icon -->

<template class="icon_th_list">
  <!-- modified from: https://fontawesome.com/icons/th-list -->
  <svg width="16" height="16" viewBox="0 0 512 512" tabindex="-1">
    <path
      fill="currentColor"
      d="M96 96c0 26.51-21.49 48-48 48S0 122.51 0 96s21.49-48 48-48 48 21.49 48 48zM48 208c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm0 160c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm96-236h352c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
      tabindex="-1"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* toc panel */
    #toc_panel {
      box-sizing: border-box;
      position: fixed;
      top: 0;
      left: 0;
      background: #ffffff;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      z-index: 2;
    }

    /* toc panel when closed */
    #toc_panel[data-open="false"] {
      min-width: 60px;
      width: 60px;
      height: 60px;
      border-right: solid 1px #bdbdbd;
      border-bottom: solid 1px #bdbdbd;
    }

    /* toc panel when open */
    #toc_panel[data-open="true"] {
      min-width: 260px;
      max-width: 480px;
      /* keep panel edge consistent distance away from "page" edge */
      width: calc(((100vw - 8.5in) / 2) - 30px - 40px);
      bottom: 0;
      border-right: solid 1px #bdbdbd;
    }

    /* toc panel header */
    #toc_header {
      box-sizing: border-box;
      display: flex;
      flex-direction: row;
      align-items: center;
      height: 60px;
      margin: 0;
      padding: 20px;
    }

    /* toc panel header when hovered */
    #toc_header:hover {
      cursor: pointer;
    }

    /* toc panel header when panel open */
    #toc_panel[data-open="true"] > #toc_header {
      border-bottom: solid 1px #bdbdbd;
    }

    /* toc open/close header button */
    #toc_button {
      margin-right: 20px;
    }

    /* hide toc list and header text when closed */
    #toc_panel[data-open="false"] > #toc_header > *:not(#toc_button),
    #toc_panel[data-open="false"] > #toc_list {
      display: none;
    }

    /* toc list of entries */
    #toc_list {
      box-sizing: border-box;
      width: 100%;
      padding: 20px;
      position: absolute;
      top: calc(60px + 1px);
      bottom: 0;
      overflow: auto;
    }

    /* toc entry, link to section in document */
    .toc_link {
      display: block;
      padding: 5px;
      position: relative;
      font-weight: 600;
      text-decoration: none;
    }

    /* toc entry when hovered or when "viewed" */
    .toc_link:hover,
    .toc_link[data-viewing="true"] {
      background: #f5f5f5;
    }

    /* toc entry, level 1 indentation */
    .toc_link[data-level="1"] {
      margin-left: 0;
    }

    /* toc entry, level 2 indentation */
    .toc_link[data-level="2"] {
      margin-left: 20px;
    }

    /* toc entry, level 3 indentation */
    .toc_link[data-level="3"] {
      margin-left: 40px;
    }

    /* toc entry, level 4 indentation */
    .toc_link[data-level="4"] {
      margin-left: 60px;
    }

    /* toc entry bullets */
    #toc_panel[data-bullets="true"] .toc_link[data-level]:before {
      position: absolute;
      left: -15px;
      top: -1px;
      font-size: 1.5em;
    }

    /* toc entry, level 2 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="2"]:before {
      content: "\2022";
    }

    /* toc entry, level 3 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="3"]:before {
      content: "\25AB";
    }

    /* toc entry, level 4 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="4"]:before {
      content: "-";
    }
  }

  /* when on screen < 8.5in wide */
  @media only screen and (max-width: 8.5in) {
    /* push <body> ("page") element down to make room for toc icon */
    .toc_body_nudge {
      padding-top: 60px;
    }

    /* toc icon when panel closed and not hovered */
    #toc_panel[data-open="false"]:not(:hover) {
      background: rgba(255, 255, 255, 0.75);
    }
  }

  /* always hide toc panel on print */
  @media only print {
    #toc_panel {
      display: none;
    }
  }
</style>
<!-- 
  Tooltips Plugin

  Makes it such that when the user hovers or focuses a link to a citation or
  figure, a tooltip appears with a preview of the reference content, along with
  arrows to navigate between instances of the same reference in the document.
-->

<script type="module">
  // whether user must click off to close tooltip instead of just un-hovering
  const clickClose = "false";
  // delay (in ms) between opening and closing tooltip
  const delay = "100";

  // start script
  function start() {
    const links = getLinks();
    for (const link of links) {
      // attach hover and focus listeners to link
      link.addEventListener("mouseover", onLinkHover);
      link.addEventListener("mouseleave", onLinkUnhover);
      link.addEventListener("focus", onLinkFocus);
      link.addEventListener("touchend", onLinkTouch);
    }

    // attach mouse, key, and resize listeners to window
    window.addEventListener("mousedown", onClick);
    window.addEventListener("touchstart", onClick);
    window.addEventListener("keyup", onKeyUp);
    window.addEventListener("resize", onResize);
  }

  // when link is hovered
  function onLinkHover() {
    // function to open tooltip
    const delayOpenTooltip = function () {
      openTooltip(this);
    }.bind(this);

    // run open function after delay
    this.openTooltipTimer = window.setTimeout(delayOpenTooltip, delay);
  }

  // when mouse leaves link
  function onLinkUnhover() {
    // cancel opening tooltip
    window.clearTimeout(this.openTooltipTimer);

    // don't close on unhover if option specifies
    if (clickClose === "true") return;

    // function to close tooltip
    const delayCloseTooltip = function () {
      // if tooltip open and if mouse isn't over tooltip, close
      const tooltip = document.getElementById("tooltip");
      if (tooltip && !tooltip.matches(":hover")) closeTooltip();
    };

    // run close function after delay
    this.closeTooltipTimer = window.setTimeout(delayCloseTooltip, delay);
  }

  // when link is focused (tabbed to)
  function onLinkFocus(event) {
    openTooltip(this);
  }

  // when link is touched on touch screen
  function onLinkTouch(event) {
    // attempt to force hover state on first tap always, and trigger
    // regular link click (and navigation) on second tap
    if (event.target === document.activeElement) event.target.click();
    else {
      document.activeElement.blur();
      event.target.focus();
    }
    if (event.cancelable) event.preventDefault();
    event.stopPropagation();
    return false;
  }

  // when mouse is clicked anywhere in window
  function onClick(event) {
    closeTooltip();
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    switch (event.key) {
      // trigger click of prev button
      case "ArrowLeft":
        const prevButton = document.getElementById("tooltip_prev_button");
        if (prevButton) prevButton.click();
        break;
      // trigger click of next button
      case "ArrowRight":
        const nextButton = document.getElementById("tooltip_next_button");
        if (nextButton) nextButton.click();
        break;
      // close on esc
      case "Escape":
        closeTooltip();
        break;
    }
  }

  // when window is resized or zoomed
  function onResize() {
    closeTooltip();
  }

  // get all links of types we wish to handle
  function getLinks() {
    const queries = [];
    // exclude buttons, anchor links, toc links, etc
    const exclude =
      ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    queries.push('a[href^="#ref-"]' + exclude); // citation links
    queries.push('a[href^="#fig:"]' + exclude); // figure links
    const query = queries.join(", ");
    return document.querySelectorAll(query);
  }

  // get links with same target, get index of link in set, get total
  // same links
  function getSameLinks(link) {
    const sameLinks = [];
    const links = getLinks();
    for (const otherLink of links) {
      if (otherLink.getAttribute("href") === link.getAttribute("href"))
        sameLinks.push(otherLink);
    }

    return {
      elements: sameLinks,
      index: sameLinks.indexOf(link),
      total: sameLinks.length,
    };
  }

  // open tooltip
  function openTooltip(link) {
    // delete tooltip if it exists, start fresh
    closeTooltip();

    // make tooltip element
    const tooltip = makeTooltip(link);

    // if source couldn't be found and tooltip not made, exit
    if (!tooltip) return;

    // make navbar elements
    const navBar = makeNavBar(link);
    if (navBar) tooltip.firstElementChild.appendChild(navBar);

    // attach tooltip to page
    document.body.appendChild(tooltip);

    // position tooltip
    const position = function () {
      positionTooltip(link);
    };
    position();

    // if tooltip contains images, position again after they've loaded
    const imgs = tooltip.querySelectorAll("img");
    for (const img of imgs) img.addEventListener("load", position);
  }

  // close (delete) tooltip
  function closeTooltip() {
    const tooltip = document.getElementById("tooltip");
    if (tooltip) tooltip.remove();
  }

  // make tooltip
  function makeTooltip(link) {
    // get target element that link points to
    const source = getSource(link);

    // if source can't be found, exit
    if (!source) return;

    // create new tooltip
    const tooltip = document.createElement("div");
    tooltip.id = "tooltip";
    const tooltipContent = document.createElement("div");
    tooltipContent.id = "tooltip_content";
    tooltip.appendChild(tooltipContent);

    // make copy of source node and put in tooltip
    const sourceCopy = makeCopy(source);
    tooltipContent.appendChild(sourceCopy);

    // attach mouse event listeners
    tooltip.addEventListener("click", onTooltipClick);
    tooltip.addEventListener("mousedown", onTooltipClick);
    tooltip.addEventListener("touchstart", onTooltipClick);
    tooltip.addEventListener("mouseleave", onTooltipUnhover);

    // (for interaction with lightbox plugin)
    // transfer click on tooltip copied img to original img
    const sourceImg = source.querySelector("img");
    const sourceCopyImg = sourceCopy.querySelector("img");
    if (sourceImg && sourceCopyImg) {
      const clickImg = function () {
        sourceImg.click();
        closeTooltip();
      };
      sourceCopyImg.addEventListener("click", clickImg);
    }

    return tooltip;
  }

  // make carbon copy of html dom element
  function makeCopy(source) {
    const sourceCopy = source.cloneNode(true);

    // delete elements marked with ignore (eg anchor and jump buttons)
    const deleteFromCopy = sourceCopy.querySelectorAll('[data-ignore="true"]');
    for (const element of deleteFromCopy) element.remove();

    // delete certain element attributes
    const attributes = [
      "id",
      "data-collapsed",
      "data-selected",
      "data-highlighted",
      "data-glow",
      "class",
    ];
    for (const attribute of attributes) {
      sourceCopy.removeAttribute(attribute);
      const elements = sourceCopy.querySelectorAll("[" + attribute + "]");
      for (const element of elements) element.removeAttribute(attribute);
    }

    return sourceCopy;
  }

  // when tooltip is clicked
  function onTooltipClick(event) {
    // when user clicks on tooltip, stop click from transferring
    // outside of tooltip (eg, click off to close tooltip, or eg click
    // off to unhighlight same refs)
    event.stopPropagation();
  }

  // when tooltip is unhovered
  function onTooltipUnhover(event) {
    if (clickClose === "true") return;

    // make sure new mouse/touch/focus no longer over tooltip or any
    // element within it
    const tooltip = document.getElementById("tooltip");
    if (!tooltip) return;
    if (this.contains(event.relatedTarget)) return;

    closeTooltip();
  }

  // make nav bar to go betwen prev/next instances of same reference
  function makeNavBar(link) {
    // find other links to the same source
    const sameLinks = getSameLinks(link);

    // don't show nav bar when singular reference
    if (sameLinks.total <= 1) return;

    // find prev/next links with same target
    const prevLink = getPrevLink(link, sameLinks);
    const nextLink = getNextLink(link, sameLinks);

    // create nav bar
    const navBar = document.createElement("div");
    navBar.id = "tooltip_nav_bar";
    const text = sameLinks.index + 1 + " of " + sameLinks.total;

    // create nav bar prev/next buttons
    const prevButton = document.createElement("button");
    const nextButton = document.createElement("button");
    prevButton.id = "tooltip_prev_button";
    nextButton.id = "tooltip_next_button";
    prevButton.title =
      "Jump to the previous occurence of this item in the document [←]";
    nextButton.title =
      "Jump to the next occurence of this item in the document [→]";
    prevButton.classList.add("icon_button");
    nextButton.classList.add("icon_button");
    prevButton.innerHTML = document.querySelector(".icon_caret_left").innerHTML;
    nextButton.innerHTML =
      document.querySelector(".icon_caret_right").innerHTML;
    navBar.appendChild(prevButton);
    navBar.appendChild(document.createTextNode(text));
    navBar.appendChild(nextButton);

    // attach click listeners to buttons
    prevButton.addEventListener("click", function () {
      onPrevNextClick(link, prevLink);
    });
    nextButton.addEventListener("click", function () {
      onPrevNextClick(link, nextLink);
    });

    return navBar;
  }

  // get previous link with same target
  function getPrevLink(link, sameLinks) {
    if (!sameLinks) sameLinks = getSameLinks(link);
    // wrap index to other side if < 1
    let index;
    if (sameLinks.index - 1 >= 0) index = sameLinks.index - 1;
    else index = sameLinks.total - 1;
    return sameLinks.elements[index];
  }

  // get next link with same target
  function getNextLink(link, sameLinks) {
    if (!sameLinks) sameLinks = getSameLinks(link);
    // wrap index to other side if > total
    let index;
    if (sameLinks.index + 1 <= sameLinks.total - 1) index = sameLinks.index + 1;
    else index = 0;
    return sameLinks.elements[index];
  }

  // get element that is target of link or url hash
  function getSource(link) {
    const hash = link ? link.hash : window.location.hash;
    const id = hash.slice(1);
    let target = document.querySelector('[id="' + id + '"]');
    if (!target) return;

    // if ref or figure, modify target to get expected element
    if (id.indexOf("ref-") === 0) target = target.querySelector(":nth-child(2)");
    else if (id.indexOf("fig:") === 0) target = target.querySelector("figure");

    return target;
  }

  // when prev/next arrow button is clicked
  function onPrevNextClick(link, prevNextLink) {
    if (link && prevNextLink)
      goToElement(prevNextLink, window.innerHeight * 0.5);
  }

  // scroll to and focus element
  function goToElement(element, offset) {
    // expand accordion section if collapsed
    expandElement(element);
    const y =
      getRectInView(element).top -
      getRectInView(document.documentElement).top -
      (offset || 0);
    // trigger any function listening for "onscroll" event
    window.dispatchEvent(new Event("scroll"));
    window.scrollTo(0, y);
    document.activeElement.blur();
    element.focus();
  }

  // determine position to place tooltip based on link position in
  // viewport and tooltip size
  function positionTooltip(link, left, top) {
    const tooltipElement = document.getElementById("tooltip");
    if (!tooltipElement) return;

    // get convenient vars for position/dimensions of
    // link/tooltip/page/view
    link = getRectInPage(link);
    const tooltip = getRectInPage(tooltipElement);
    const view = getRectInPage();

    // horizontal positioning
    if (left)
      // use explicit value
      left = left;
    else if (link.left + tooltip.width < view.right)
      // fit tooltip to right of link
      left = link.left;
    else if (link.right - tooltip.width > view.left)
      // fit tooltip to left of link
      left = link.right - tooltip.width;
    // center tooltip in view
    else left = (view.right - view.left) / 2 - tooltip.width / 2;

    // vertical positioning
    if (top)
      // use explicit value
      top = top;
    else if (link.top - tooltip.height > view.top)
      // fit tooltip above link
      top = link.top - tooltip.height;
    else if (link.bottom + tooltip.height < view.bottom)
      // fit tooltip below link
      top = link.bottom;
    else {
      // center tooltip in view
      top = view.top + view.height / 2 - tooltip.height / 2;
      // nudge off of link to left/right if possible
      if (link.right + tooltip.width < view.right) left = link.right;
      else if (link.left - tooltip.width > view.left)
        left = link.left - tooltip.width;
    }

    tooltipElement.style.left = left + "px";
    tooltipElement.style.top = top + "px";
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
  <!-- modified from: https://fontawesome.com/icons/caret-left -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
    ></path>
  </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
  <!-- modified from: https://fontawesome.com/icons/caret-right -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* tooltip container */
    #tooltip {
      position: absolute;
      width: 50%;
      min-width: 240px;
      max-width: 75%;
      z-index: 1;
    }

    /* tooltip content */
    #tooltip_content {
      margin-bottom: 5px;
      padding: 20px;
      border-radius: 5px;
      border: solid 1px #bdbdbd;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      background: #ffffff;
      overflow-wrap: break-word;
    }

    /* tooltip copy of paragraphs and figures */
    #tooltip_content > p,
    #tooltip_content > figure {
      margin: 0;
      max-height: 320px;
      overflow-y: auto;
    }

    /* tooltip copy of <img> */
    #tooltip_content > figure > img,
    #tooltip_content > figure > svg {
      max-height: 260px;
    }

    /* navigation bar */
    #tooltip_nav_bar {
      margin-top: 10px;
      text-align: center;
    }

    /* navigation bar previous/next buton */
    #tooltip_nav_bar > .icon_button {
      position: relative;
      top: 3px;
    }

    /* navigation bar previous button */
    #tooltip_nav_bar > .icon_button:first-of-type {
      margin-right: 5px;
    }

    /* navigation bar next button */
    #tooltip_nav_bar > .icon_button:last-of-type {
      margin-left: 5px;
    }
  }

  /* always hide tooltip on print */
  @media only print {
    #tooltip {
      display: none;
    }
  }
</style>
<!--
  Analytics Plugin (third-party) 
  
  Copy and paste code from Google Analytics or similar service here.
-->
<!-- 
  Annotations Plugin

  Allows public annotation of the  manuscript. See https://web.hypothes.is/.
-->

<script type="module">
  // configuration
  window.hypothesisConfig = function () {
    return {
      branding: {
        accentColor: "#2196f3",
        appBackgroundColor: "#f8f8f8",
        ctaBackgroundColor: "#f8f8f8",
        ctaTextColor: "#000000",
        selectionFontFamily: "Open Sans, Helvetica, sans serif",
        annotationFontFamily: "Open Sans, Helvetica, sans serif",
      },
    };
  };

  // hypothesis client script
  const embed = "https://hypothes.is/embed.js";
  // hypothesis annotation count query url
  const query = "https://api.hypothes.is/api/search?limit=0&url=";

  // start script
  function start() {
    const button = makeButton();
    document.body.insertBefore(button, document.body.firstChild);
    insertCount(button);
  }

  // make button
  function makeButton() {
    // create button
    const button = document.createElement("button");
    button.id = "hypothesis_button";
    button.innerHTML = document.querySelector(".icon_hypothesis").innerHTML;
    button.title = "Hypothesis annotations";
    button.classList.add("icon_button");

    function onClick(event) {
      onButtonClick(event, button);
    }

    // attach click listeners
    button.addEventListener("click", onClick);

    return button;
  }

  // insert annotations count
  async function insertCount(button) {
    // get annotation count from Hypothesis based on url
    let count = "-";
    try {
      const canonical = document.querySelector('link[rel="canonical"]');
      const location = window.location;
      const url = encodeURIComponent((canonical || location).href);
      const response = await fetch(query + url);
      const json = await response.json();
      count = json.total || "-";
    } catch (error) {
      console.log(error);
    }

    // put count into button
    const counter = document.createElement("span");
    counter.id = "hypothesis_count";
    counter.innerHTML = count;
    button.title = "View " + count + " Hypothesis annotations";
    button.append(counter);
  }

  // when button is clicked
  function onButtonClick(event, button) {
    const script = document.createElement("script");
    script.src = embed;
    document.body.append(script);
    button.remove();
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- hypothesis icon -->

<template class="icon_hypothesis">
  <!-- modified from: https://simpleicons.org/icons/hypothesis.svg / https://git.io/Jf1VB -->
  <svg width="16" height="16" viewBox="0 0 24 24" tabindex="-1">
    <path
      fill="currentColor"
      d="M3.43 0C2.5 0 1.72 .768 1.72 1.72V18.86C1.72 19.8 2.5 20.57 3.43 20.57H9.38L12 24L14.62 20.57H20.57C21.5 20.57 22.29 19.8 22.29 18.86V1.72C22.29 .77 21.5 0 20.57 0H3.43M5.14 3.43H7.72V9.43S8.58 7.72 10.28 7.72C12 7.72 13.74 8.57 13.74 11.24V17.14H11.16V12C11.16 10.61 10.28 10.07 9.43 10.29C8.57 10.5 7.72 11.41 7.72 13.29V17.14H5.14V3.43M18 13.72C18.95 13.72 19.72 14.5 19.72 15.42A1.71 1.71 0 0 1 18 17.13A1.71 1.71 0 0 1 16.29 15.42C16.29 14.5 17.05 13.71 18 13.71Z"
      tabindex="-1"
    ></path>
  </svg>
</template>

<style>
  /* hypothesis activation button */
  #hypothesis_button {
    box-sizing: border-box;
    position: fixed;
    top: 0;
    right: 0;
    width: 60px;
    height: 60px;
    background: #ffffff;
    border-radius: 0;
    border-left: solid 1px #bdbdbd;
    border-bottom: solid 1px #bdbdbd;
    box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
    z-index: 2;
  }

  /* hypothesis button svg */
  #hypothesis_button > svg {
    position: relative;
    top: -4px;
  }

  /* hypothesis annotation count */
  #hypothesis_count {
    position: absolute;
    left: 0;
    right: 0;
    bottom: 5px;
  }

  /* side panel */
  .annotator-frame {
    width: 280px !important;
  }

  /* match highlight color to rest of theme */
  .annotator-highlights-always-on .annotator-hl {
    background-color: #ffeb3b !important;
  }

  /* match focused color to rest of theme */
  .annotator-hl.annotator-hl-focused {
    background-color: #ff8a65 !important;
  }

  /* match bucket bar color to rest of theme */
  .annotator-bucket-bar {
    background: #f5f5f5 !important;
  }

  /* always hide button, toolbar, and tooltip on print */
  @media only print {
    #hypothesis_button {
      display: none;
    }

    .annotator-frame {
      display: none !important;
    }

    hypothesis-adder {
      display: none !important;
    }
  }
</style>
<!-- 
  Mathjax Plugin (third-party) 

  Allows the proper rendering of math/equations written in LaTeX.
  See https://www.mathjax.org/.
-->

<script type="text/x-mathjax-config">
  // configuration
  MathJax.Hub.Config({
    "CommonHTML": { linebreaks: { automatic: true } },
    "HTML-CSS": { linebreaks: { automatic: true } },
    "SVG": { linebreaks: { automatic: true } },
    "fast-preview": { disabled: true }
  });
</script>

<script
  src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"
  integrity="sha512-0doc9hKxR3PYwso42RD1p5ySZpzzuDiOwMrdCEh2WdJZCjcmFKc/wEnL+z8fBQrnHoiNWbo+3fiGkOYXBdQp4A=="
  crossorigin="anonymous"
></script>

<style>
  /* mathjax containers */
  .math.display > span:not(.MathJax_Preview) {
    /* turn inline element (no dimensions) into block (allows fixed width and thus scrolling) */
    display: flex !important;
    overflow-x: auto !important;
    overflow-y: hidden !important;
    justify-content: center;
    align-items: center;
    margin: 0 !important;
  }

  /* right click menu */
  .MathJax_Menu {
    border-radius: 5px !important;
    border: solid 1px #bdbdbd !important;
    box-shadow: none !important;
  }

  /* equation auto-number */
  span[id^="eq:"] > span.math.display + span {
    font-weight: 600;
  }

  /* equation */
  span[id^="eq:"] > span.math.display > span {
    /* nudge to make room for equation auto-number and anchor */
    margin-right: 60px !important;
  }
</style>
</body>
</html>
